| Term                                                         | Category                      | Source/Section Reference in Manual |
| :----------------------------------------------------------- | :---------------------------- | :--------------------------------- |
| Amber 2025                                                   | Software Suite                | Cover, Page 3, 4, 1. Introduction  |
| Amber24                                                      | Software Suite                | Cover, Page 3, 4, 1. Introduction  |
| AmberTools25                                                 | Software Suite                | Cover, Page 3, 4, 1. Introduction  |
| ZIP metal transporter                                        | Chemical Entity               | Page 4 (Cover Illustration)        |
| inward-facing state                                          | State                         | Page 4 (Cover Illustration)        |
| outward-facing state                                         | State                         | Page 4 (Cover Illustration)        |
| Cl- ions                                                     | Chemical Entity               | Page 4 (Cover Illustration)        |
| K+ ions                                                      | Chemical Entity               | Page 4 (Cover Illustration)        |
| zinc ion                                                     | Chemical Entity               | Page 4 (Cover Illustration)        |
| enhanced sampling simulations                                | Method                        | Page 4 (Cover Illustration)        |
| elevator-like motions                                        | Process                       | Page 4 (Cover Illustration)        |
| hinge motion                                                 | Process                       | Page 4 (Cover Illustration)        |
| vertical sliding                                             | Process                       | Page 4 (Cover Illustration)        |
| cmake build system                                           | Software/Tool                 | 2. Installation, 2.2               |
| Amber force fields                                           | Model                         | II. Amber force fields, 3.           |
| Molecular mechanics force fields                             | Model                         | 3.                                 |
| Proteins                                                     | Chemical Entity Class         | 3.1                                |
| Nucleic acids                                                | Chemical Entity Class         | 3.2                                |
| Modified amino acids and nucleotides                         | Chemical Entity Class         | 3.3                                |
| Carbohydrates                                                | Chemical Entity Class         | 3.4                                |
| Lipids                                                       | Chemical Entity Class         | 3.5                                |
| Solvents                                                     | Chemical Entity Class         | 3.6                                |
| Ions                                                         | Chemical Entity Class         | 3.7                                |
| semi-empirical QM                                            | Method/Model                  | 3.8, 9.                            |
| GAL17 force field                                            | Model                         | 3.9                                |
| AMBER-DYES                                                   | Chemical Entity/Model         | 3.10                               |
| SIRAH force field                                            | Model                         | 3.11                               |
| Generalized Born/Surface Area Model (GB/SA)                  | Model                         | 4.                                 |
| GB/SA input parameters                                       | Data/Parameter                | 4.1                                |
| Implicit Solvent with Explicit Ions model (GBION)            | Model                         | 4.2                                |
| Group Transfer Free Energy (GTFE) model                      | Model                         | 4.3                                |
| ALPB (Analytical Linearized Poisson-Boltzmann)               | Model/Method                  | 4.4                                |
| GBNSR6                                                       | Model/Algorithm               | 5.                                 |
| PBSA (Poisson-Boltzmann Surface Area)                        | Model/Algorithm               | 6.                                 |
| Torch PBSA                                                   | Software/Model                | 7.                                 |
| LibTorch                                                     | Software/Library              | 7.1                                |
| Reference Interaction Site Model (RISM)                      | Model                         | 8.                                 |
| 1D-RISM                                                      | Method/Implementation         | 8.1.1, 8.4                         |
| 3D-RISM                                                      | Method/Implementation         | 8.1.2, 8.5, 8.6                    |
| sqm                                                          | Software                      | 9.                                 |
| NDDO (Neglect of Diatomic Differential Overlap) methods      | Method                        | 9., 11.1                           |
| SCC-DFTB (Self-Consistent Charge Density Functional Tight Binding) | Method                        | 9., 11.1                           |
| QUICK                                                        | Software                      | 10.                                |
| ab initio quantum chemistry                                  | Method Class                  | 10.                                |
| QM/MM calculations                                           | Method                        | 11.                                |
| TeraChem                                                     | Software                      | 11., 11.2, 11.4                    |
| xTB                                                          | Software                      | 11., 11.5                          |
| DFTB+                                                        | Software                      | 11., 11.6                          |
| DPRc (Deep Potential Range Corrected) machine learning corrections | Method/Model                  | 11.7                               |
| Adaptive solvent QM/MM simulations                           | Method                        | 11.8                               |
| SEBOMD (SemiEmpirical Born-Oppenheimer Molecular Dynamics)   | Method                        | 11.10                              |
| ReaxFF/AMBER                                                 | Model/Method                  | 11.11                              |
| PDB Files / PDB format                                       | Data Format                   | 1.1, 13.                           |
| Residue naming conventions                                   | Convention                    | 13.2                               |
| pdb4amber                                                    | Software/Tool                 | 1.1, 13.4                          |
| reduce (program)                                             | Software/Tool                 | 13.5                               |
| pdb2pqr                                                      | Software/Tool                 | 13.6                               |
| gwh (guess water hydrogens)                                  | Software/Tool                 | 13.7                               |
| packmol-memgen                                               | Software/Tool                 | 1.1.1, 13.9                        |
| AMBAT                                                        | Software/Tool                 | 13.10                              |
| LEaP (tleap, xleap)                                          | Software/Tool                 | 1.1, 1.1.1, 14.                    |
| Amber parameter files                                        | Data Format                   | 15.                                |
| ParmEd                                                       | Software/Tool                 | 1.1, 15.2                          |
| Antechamber                                                  | Software/Tool                 | 1.1, 16.                           |
| GAFF (General Amber Force Field)                             | Model                         | 1.1.1, 16.                         |
| paramfit                                                     | Software/Tool                 | 17.                                |
| mdgx                                                         | Software/Tool                 | 1.1.1, 18.                         |
| pyMSMT (Python Metal Site Modeling Toolbox)                  | Software/Tool                 | 1.1.1, 19.                         |
| py_resp.py                                                   | Software/Tool                 | 20.                                |
| PyPE_RESP                                                    | Method/Software               | 20.4                               |
| UnitCell                                                     | Software/Tool                 | 21.1                               |
| PropPDB                                                      | Software/Tool                 | 21.2                               |
| AddToBox                                                     | Software/Tool                 | 21.3                               |
| ChBox                                                        | Software/Tool                 | 21.4                               |
| sander                                                       | Software                      | 1.1, 1.1.2, 22.                    |
| Namelist Input Syntax                                        | Data Format/Convention        | 22.4                               |
| Polarizable Gaussian Multipole Model                         | Model                         | 22.8                               |
| multisander                                                  | Software/Method               | 22.12                              |
| multipmemd                                                   | Software/Method               | 22.12                              |
| APBS (Adaptive Poisson-Boltzmann Solver)                     | Software                      | 22.13                              |
| pmemd                                                        | Software                      | 1. Introduction, 1.1.2, 23.        |
| GPU Accelerated PMEMD                                        | Method/Software Feature       | 23.6                               |
| Amber Masks                                                  | Convention/Syntax             | 24.1                               |
| Atom Expressions (NAB)                                       | Convention/Syntax             | 24.2                               |
| GROUP Specification                                          | Convention/Syntax             | 24.3                               |
| Self-Guided Langevin dynamics                                | Method/Algorithm              | 25.1                               |
| Accelerated Molecular Dynamics                               | Method/Algorithm              | 25.2                               |
| Gaussian Accelerated Molecular Dynamics                      | Method/Algorithm              | 25.3                               |
| Targeted MD                                                  | Method/Algorithm              | 25.4                               |
| Multiply-Targeted MD (MTMD)                                  | Method/Algorithm              | 25.5                               |
| Nudged elastic band calculations                             | Method/Algorithm              | 25.6                               |
| Adaptive String Method                                       | Method/Algorithm              | 25.7                               |
| Low-MODe (LMOD) methods                                      | Method/Algorithm              | 25.8                               |
| DL-Find Optimization                                         | Method/Algorithm              | 25.9                               |
| Thermodynamic integration (TI)                               | Method/Concept                | 1.1.2, 26.1, 40.6                  |
| Linear Interaction Energies (LIE)                            | Method/Concept                | 26.2, 40.5                         |
| Replica Exchange Molecular Dynamics (REMD)                   | Method/Algorithm              | 26.3                               |
| Adaptively Biased MD                                         | Method/Algorithm              | 26.4                               |
| Steered MD (SMD)                                             | Method/Algorithm              | 26.4, 26.5                         |
| Umbrella Sampling                                            | Method/Algorithm              | 26.4                               |
| String Method (for free energies)                            | Method/Algorithm              | 26.4                               |
| Jarzynski Relationship/Equation                              | Concept/Equation              | 26.5, 26.7                         |
| EMIL (Absolute Free Energies using EMIL)                     | Method                        | 26.6                               |
| Constant pH calculations                                     | Method                        | 27.                                |
| cphstats                                                     | Software/Tool                 | 27.7                               |
| Constant Redox Potential calculations                        | Method                        | 28.                                |
| cestats                                                      | Software/Tool                 | 28.6                               |
| Continuous constant pH molecular dynamics                    | Method                        | 29.                                |
| NMR refinement                                               | Process/Method                | 30.                                |
| NOESY volume restraints                                      | Data/Constraint               | 30.2                               |
| Chemical shift restraints                                    | Data/Constraint               | 30.3                               |
| Pseudocontact shift restraints                               | Data/Constraint               | 30.4                               |
| Direct dipolar coupling restraints                           | Data/Constraint               | 30.5                               |
| Residual CSA or pseudo-CSA restraints                        | Data/Constraint               | 30.6                               |
| Xray and cryoEM refinement                                   | Process/Method                | 31.                                |
| EMAP restraints                                              | Data/Constraint/Method        | 31.1                               |
| FRETrest (Forster Resonance Energy Transfer restraints)      | Data/Constraint/Method        | 31.2                               |
| Locally-enchanced sampling (LES)                             | Method                        | 32.                                |
| ADDLES program                                               | Software                      | 32.2                               |
| gem.pmemd                                                    | Software                      | 1.1.2, 33.                         |
| KMMD (Molecular Dynamics Using a Kernel Machine)             | Method                        | 34.1                               |
| NAB, nabc, libsff                                            | Software/Library              | 35.                                |
| mdout_analyzer.py                                            | Software/Tool                 | 1.1.3, 36.                         |
| ambpdb                                                       | Software/Tool                 | 36.1                               |
| cpptraj                                                      | Software/Tool                 | 1.1.1, 1.1.3, 37.                  |
| pytraj                                                       | Software/Library              | 1.1.3, 38.                         |
| MMPBSA.py                                                    | Software/Method               | 1.1.3, 39.                         |
| FEW (Free energy workflow)                                   | Software/Workflow             | 1.1.3, 40.                         |
| WAMM (Workflow for automated MM-PBSA & MM-GBSA)              | Workflow/Method               | 40.4                               |
| LIEW (Linear interaction energy workflow)                    | Workflow/Method               | 40.5                               |
| TIW (Thermodynamic integration workflow)                     | Workflow/Method               | 40.6                               |
| BAR/PBSA                                                     | Method/Software               | 41.                                |
| edgembar                                                     | Software/Method               | 1.1.3, 42.                         |
| fetkutils-tischedule.py                                      | Software                      | 42.5                               |
| SAXS (Small-Angle X-ray Scattering)                          | Method                        | 43.                                |
| MoFT (analysis of volumetric data)                           | Software/Method               | 44.                                |
| ndfes                                                        | Software/Method               | 1.1.3, 45.                         |
| Cartesian coordinates                                        | Data Type                     | 1.1                                |
| Residue libraries                                            | Data                          | 1.1                                |
| atom names                                                   | Data/Parameter                | 1.1                                |
| atom types                                                   | Data/Parameter                | 1.1                                |
| residue names                                                | Data/Parameter                | 1.1                                |
| charges (partial atomic charges)                             | Data/Parameter                | 1.1                                |
| $AMBERHOME/dat/leap/lib                                      | File Path/Convention          | 1.1                                |
| $AMBERHOME/dat/leap/parm                                     | File Path/Convention          | 1.1                                |
| prmtop                                                       | File Format                   | 1.1                                |
| prmcrd                                                       | File Format                   | 1.1                                |
| mdin                                                         | File Format                   | 1.1                                |
| AMBERHOME                                                    | Environment Variable/Convention | 2.1, Page 20                       |
| MPI (Message Passing Interface)                              | Software/Standard             | 2.1, 2.2.1                         |
| OpenMP                                                       | Software/Standard             | 2.2.1                              |
| CUDA                                                         | Software/Platform             | 2.2.2                              |
| HIP (Heterogeneous-Compute Interface for Portability)        | Software/API                  | 2.2.3                              |
| ROCm                                                         | Software Stack                | 2.2.3                              |
| NetCDF                                                       | Software/Library/Data Format  | 2.2.4                              |
| FFTW                                                         | Software/Library              | 2.2.4                              |
| BLAS (Basic Linear Algebra Subprograms)                      | Software/Library/Standard     | 2.2.5                              |
| LAPACK (Linear Algebra PACKage)                              | Software/Library/Standard     | 2.2.5                              |
| MKL (Math Kernel Library)                                    | Software/Library              | 2.2.4, 2.2.5                       |
| Python                                                       | Programming Language          | 2.2.6                              |
| ff19SB                                                       | Model                         | 3.1, 3.1.1                         |
| OL24                                                         | Model                         | 3.1, 3.2.2                         |
| OL3                                                          | Model                         | 3.1, 3.2.1                         |
| GLYCAM_06j                                                   | Model                         | 3.1, 3.4                           |
| lipid21                                                      | Model                         | 3.1, 3.5.1                         |
| spc/e water model                                            | Model                         | 3.1, 3.6                           |
| tip4pew water model                                          | Model                         | 3.1, 3.6                           |
| OPC water model                                              | Model                         | 3.1, 3.1.1, 3.6, 3.6.1             |
| gaff2                                                        | Model                         | 3.1                                |
| TIP3P water model                                            | Model                         | 3.1 (note 2), 3.6                  |
| CMAPs (Correction Maps)                                      | Model Feature/Parameter       | 3.1.1                              |
| ff14SB                                                       | Model                         | 3.1.1                              |
| ff15ipq                                                      | Model                         | 3.1.2                              |
| IPolQ charge derivation                                      | Method                        | 3.1.2                              |
| fb15 (force balance protein force field)                     | Model                         | 3.1.3                              |
| ff03 (Duan et al. 2003 force field)                          | Model                         | 3.1.4                              |
| ff03ua (Yang et al. 2003 united-atom force field)            | Model                         | 3.1.5                              |
| Intrinsically disordered proteins (IDPs)                     | Chemical Entity Class         | 3.1.6                              |
| ff99-bsc0                                                    | Model/Parameter Set           | 3.2                                |
| OHE C3 (single nucleotide naming)                            | Convention                    | 3.2.3                              |
| modXNA                                                       | Software/Tool                 | 3.3.4                              |
| GLYCAM06EP                                                   | Model                         | 3.4, 3.4.5                         |
| GLYCAM Naming Convention                                     | Convention                    | 3.4.6                              |
| LIPID11, LIPID14, LIPID17                                    | Model                         | 3.5                                |
| NPT ensemble                                                 | Simulation Condition          | 3.5                                |
| Berendsen barostat                                           | Algorithm/Method              | 3.5                                |
| Monte Carlo Barostat                                         | Algorithm/Method              | 3.5                                |
| OPC3 water model                                             | Model                         | 3.6, 3.6.1                         |
| OPC3-pol water model                                         | Model                         | 3.6.1.1                            |
| Drude oscillator                                             | Model/Concept                 | 3.6.1.1, 3.9                       |
| HMR (Hydrogen Mass Repartitioning)                           | Method                        | 3.6.1.1                            |
| Joung/Cheatham ion parameters                                | Parameter Set                 | 3.1.3, 3.6, 3.7, 4.2.1             |
| Li/Merz ion parameters                                       | Parameter Set                 | 3.6.1, 3.7                         |
| 12-6 LJ nonbonded model                                      | Model Type                    | 3.7                                |
| 12-6-4 LJ-type nonbonded model                               | Model Type                    | 3.7                                |
| ParmAM1, parmPM3                                             | Model/Parameter Set           | 3.8                                |
| AM1, PM3 (semi-empirical methods)                            | Method                        | 3.8                                |
| Pt(111) surface                                              | Material                      | 3.9                                |
| &music namelist                                              | Namelist                      | 3.9                                |
| belly dynamics                                               | Method/Technique              | 3.9                                |
| FRET (Forster Resonance Energy Transfer)                     | Method/Phenomenon             | 3.3.2, 3.10, 31.2                  |
| Fluorescent protein chromophores (eGFP, CRO etc.)            | Chemical Entity Class         | 3.10.1                             |
| Coarse-Grained (CG) force field                              | Model Type                    | 3.11                               |
| WatFour (WT4) water model                                    | Model                         | 3.11                               |
| cgconv.pl                                                    | Software/Script               | 3.11.2                             |
| leaprc.sirah                                                 | File/Command                  | 3.11.2                             |
| effective Born radii (Ri)                                    | Parameter                     | 4.                                 |
| Debye-Huckel screening parameter (κ)                         | Parameter                     | 4.                                 |
| Coulomb field approximation (CFA)                            | Approximation/Method          | 4.                                 |
| OBC model (igb=2, igb=5)                                     | Model                         | 4., 4.1                            |
| GBn model (igb=7, igb=8)                                     | Model                         | 4.1                                |
| mbondi, mbondi2, mbondi3, bondi (PBRadii sets)               | Parameter Set                 | 4.1, Table 4.1                     |
| LCPO algorithm                                               | Algorithm                     | 4., 4.1, 4.3                       |
| KGB(a,b), Kε(a,b) (GBION coefficients)                       | Parameter                     | 4.2                                |
| SLTCAP method                                                | Method                        | 4.2.1                              |
| R6 integration (GBNSR6)                                      | Method                        | 5.                                 |
| CHAGB (Charge Hydration Asymmetric Generalized Born)         | Model                         | 5.1                                |
| field-view method (molecular surface)                        | Method                        | 5.2                                |
| finite-difference (FD) method                                | Algorithm                     | 6.1.1                              |
| Immersed Interface method (IIM)                              | Algorithm                     | 6.1.1, 6.7.2                       |
| SES (Solvent Excluded Surface)                               | Surface Definition            | 6.1.3, 6.2.2, 7.2.1                |
| SAS (Solvent Accessible Surface)                             | Surface Definition            | 6.2.2                              |
| MLSES (Machine-Learned Solvent Excluded Surface)             | Model                         | 6.2.2, 6.2.9, 7.2.1                |
| GENIUSES                                                     | Model/Algorithm               | 6.2.9, 7.2.1, 7.2.1.1              |
| Con2SES                                                      | Model/Algorithm               | 6.2.9, 7.2.1, 7.2.1.2              |
| Active site focusing (PBSA)                                  | Method                        | 6.2.10                             |
| BiCG (biconjugate gradient) solver                           | Algorithm                     | 6.7                                |
| Ornstein-Zernike (OZ) equation                               | Equation/Theory               | 8.1                                |
| HNC, KH, PSE-n, MSA (RISM Closures)                          | Closure/Approximation         | 8.1.1, 8.1.2, 8.4.1, 8.5.2.1       |
| DRISM (Dielectrically consistent RISM)                       | Method/Theory                 | 8.1.1, 8.4.1                       |
| MDIIS (Modified Direct Inversion of the Iterative Subspace)  | Algorithm                     | 8.1.1, 8.3.4.2, 8.5.2.1            |
| Treecode Summation (RISM)                                    | Algorithm/Method              | 8.1.4                              |
| Molecular Reconstruction (RISM)                              | Method/Analysis               | 8.1.5, 8.5.2.2                     |
| Universal Correction (UC), Ng Bridge Correction (NgB), Pressure Correction Plus (PC+) (RISM Solvation Corrections) | Method/Correction Term        | 8.2.4, 8.5.2.1                     |
| r-RESPA (for RISM)                                           | Algorithm                     | 8.5.1                              |
| Force-coordinate extrapolation (FCE) (for RISM)              | Method                        | 8.5.1                              |
| ASFE, GSFE extrapolation (RISM FCE variants)                 | Method                        | 8.5.2.1                            |
| MDL file format (RISM)                                       | Data Format                   | 8.7.1                              |
| XVV file format (RISM)                                       | Data Format                   | 8.7.2                              |
| MRC 2014 format, CCP4 format, DX format, XYZV format (Volumetric Data) | Data Format                   | 8.5.2.2, 8.7.6, 8.7.7, 8.7.8       |
| AM1-bcc charges                                              | Parameter Type/Method         | 9.                                 |
| PM3, AM1, RM1, MNDO, PDDG/PM3, PDDG/MNDO, PM3CARB1, PM3-MAIS, MNDO/d, AM1/d, PM6 (Semi-empirical Hamiltonians) | Method                        | 9.1                                |
| mio-1-1, 3ob-3-1 (DFTB parameter sets)                       | Parameter Set                 | 9.1, 9.1.1                         |
| CM3 charges (DFTB2)                                          | Parameter Type/Method         | 9.1                                |
| Empirical dispersion correction (DFT-D like)                 | Method/Correction             | 9.2                                |
| Empirical hydrogen bonding correction                        | Method/Correction             | 9.2                                |
| Slater-Koster files (.skf)                                   | File Format                   | 9.2 (implicit)                     |
| Hartree-Fock (HF)                                            | Method                        | 10., 11.2.6.2, 11.2.6.3, 11.2.6.5 |
| Density Functional Theory (DFT)                              | Method                        | 10., 11.2, 11.2.6.1, ...           |
| Mulliken charges                                             | Parameter Type                | 10.1, 11.1, 11.1.4                 |
| Molden format                                                | Data Format                   | 10.1                               |
| Gaussian basis sets                                          | Basis Set Type                | 10.1                               |
| SG1 grid (DFT)                                               | Grid Type                     | 10.1                               |
| Link atom approach (QM/MM)                                   | Method                        | 11.1.1, 11.2                       |
| PM3/MM*, PM3/MMX2 (Reformulated PM3 QM/MM interface)         | Model/Method                  | 11.1.2                             |
| qxd (Charge-dependent exchange-dispersion corrections)       | Method/Correction             | 11.1.8                             |
| TCPB-cpp (TeraChem Protocol Buffers client)                  | Software/Interface            | 11., 11.4, 11.4.1.1                |
| GFN1-xTB, GFN2-xTB Hamiltonians                              | Method/Model                  | 11.5                               |
| Grimme’s D3 dispersion with Becke-Johnson damping            | Method/Correction             | 11.6.1                             |
| D3H5, D42B, D43B corrections (DFTB3)                         | Method/Correction             | 11.6.1                             |
| DeePMD-kit                                                   | Software                      | 11.7                               |
| QM/MM+∆MLP model                                             | Model Type                    | 11.7                               |
| DP-GEN, DP-Amber (DeePMD-kit utilities)                      | Software/Utility              | 11.7.3                             |
| adQM/MM (adaptive QM/MM)                                     | Method                        | 11.8                               |
| DAS (difference-based adaptive solvation) method              | Method                        | 11.8                               |
| λ(r) (adQM/MM partial MM character switching function)       | Function/Parameter            | 11.8.1.1                           |
| Force interpolation (adQM/MM)                                | Method                        | 11.8.1.2                           |
| multisander (for adQM/MM)                                    | Software Architecture         | 11.8.2                             |
| Adaptive solvent QM/MM simulations                           | Method                        | 11.8                               |
| adQM/MM (adaptive QM/MM)                                     | Method                        | 11.8                               |
| A region (active region)                                     | Region/Concept                | 11.8.1                             |
| T region (transition region)                                 | Region/Concept                | 11.8.1                             |
| E region (environment region)                                | Region/Concept                | 11.8.1                             |
| λ(r) (partial MM character switching function)               | Function/Parameter            | 11.8.1.1                           |
| Force interpolation (adQM/MM)                                | Method                        | 11.8.1.2                           |
| multisander                                                  | Software Feature              | 11.8.2                             |
| groupfile (for multisander)                                  | File Format                   | 11.8.2                             |
| &vsolv namelist                                              | Namelist                      | 11.8.2.2                           |
| &adqmmm namelist                                             | Namelist                      | 11.8.2.2                           |
| Adaptive buffered force-mixing QM/MM (abfQM/MM)              | Method                        | 11.9                               |
| core QM region                                               | Region/Concept                | 11.9.2                             |
| dynamical QM region                                          | Region/Concept                | 11.9.2                             |
| buffer region                                                | Region/Concept                | 11.9.2                             |
| extended system (abfQM/MM)                                   | System Definition             | 11.9.4.1                           |
| reduced system (abfQM/MM)                                    | System Definition             | 11.9.4.1                           |
| core atoms (abfQM/MM)                                        | Atom Type (abfQM/MM)          | 11.9.4.2                           |
| qm atoms (abfQM/MM)                                          | Atom Type (abfQM/MM)          | 11.9.4.2                           |
| buffer atoms (abfQM/MM)                                      | Atom Type (abfQM/MM)          | 11.9.4.2                           |
| mm atoms (abfQM/MM)                                          | Atom Type (abfQM/MM)          | 11.9.4.2                           |
| &qmmm namelist (for abfQM/MM)                                | Namelist                      | 11.9.5                             |
| ambmask format                                               | Data Format/Syntax            | 11.9.5.1                           |
| Adaptive thermostats (Nose-Hoover, Langevin)                 | Algorithm/Method              | 11.9.5.2                           |
| SEBOMD (SemiEmpirical Born-Oppenheimer Molecular Dynamics)   | Method/Software Feature       | 11.10                              |
| DivCon                                                       | Software                      | 11.10                              |
| &sebomd namelist                                             | Namelist                      | 11.10.2, 11.10.3                   |
| PIF2, PIF3 (PM3 corrections)                                 | Correction/Model              | 11.10.3 (modif)                    |
| MAIS1, MAIS2 (PM3 corrections)                               | Correction/Model              | 11.10.3 (modif)                    |
| ReaxFF/AMBER                                                 | Method/Software Feature       | 11.11                              |
| PuReMD (Reactive Molecular Dynamics)                         | Software                      | 11.11                              |
| ReaxFF force fields                                          | Model                         | 11.11.2                            |
| QEq (Charge equilibration)                                   | Method                        | 11.11.5 (charge_method)            |
| EE (Electronegativity equalization)                          | Method                        | 11.11.5 (charge_method)            |
| ACKS2 (Atom-condensed Kohn-Sham approximated to second order) | Method                        | 11.11.5 (charge_method)            |
| &reaxff namelist                                             | Namelist                      | 11.11.5                            |
| reaxff-puremd control file                                   | File Format                   | 11.11.6                            |
| External library interface (Amber)                           | Software Feature              | 12.                                |
| MBX library                                                  | Software/Library              | 12.                                |
| KMMD library (Machine Learning Molecular Dynamics)           | Software/Library              | 12.                                |
| &extpot namelist                                             | Namelist                      | 12.2                               |
| PDB Files (Preparation)                                      | Process                       | 13.                                |
| pdb4amber                                                    | Software/Tool                 | 13.                                |
| reduce (program for adding hydrogens)                        | Software/Tool                 | 13., 13.5                          |
| TER record (PDB)                                             | Data Format Element           | 13.1, 13.2, 13.3                   |
| HIE, HID, HIP (Histidine protonation states)                 | Chemical State/Convention     | 13.2                               |
| CYX (Cysteine in disulfide bridge)                           | Chemical State/Convention     | 13.2                               |
| ASH, GLH, LYN (Neutral Asp, Glu, Lys)                        | Chemical State/Convention     | 13.2                               |
| ACE, NHE, NME (Terminal cap residues)                        | Chemical Entity/Convention    | 13.2                               |
| AMBER masks                                                  | Syntax/Convention             | 13.3                               |
| NAB atom expressions                                         | Syntax/Convention             | 13.3                               |
| pytleap                                                      | Software/Script               | 13.3                               |
| pdb2pqr                                                      | Software/Tool                 | 13.6                               |
| propka3                                                      | Software (used by pdb2pqr)    | 13.6                               |
| gwh (guess water hydrogens)                                  | Software/Tool                 | 13.7                               |
| packmol-memgen                                               | Software/Tool                 | 13.9                               |
| Memembed                                                     | Software (used by packmol-memgen) | 13.9                               |
| pdbremix                                                     | Software (used by packmol-memgen) | 13.9                               |
| Packmol                                                      | Software (used by packmol-memgen) | 13.9                               |
| charmmlipid2amber.py                                         | Software/Script               | 13.9                               |
| OPM/PPM3 orientation program                                 | Software                      | 13.9                               |
| immers                                                       | Software                      | 13.9                               |
| Lipid ext (Lipid extension force field)                      | Model                         | 13.9.1                             |
| lysophospholipids                                            | Chemical Entity Class         | 13.9.1                             |
| phosphatidylinositol                                         | Chemical Entity Class         | 13.9.1                             |
| cardiolipins                                                 | Chemical Entity Class         | 13.9.1                             |
| sterols (ergosterol, campesterol, sitosterol, stigmasterol)  | Chemical Entity Class         | 13.9.1                             |
| memgen.parm file                                             | File Format                   | 13.9.2                             |
| SIRAH force field (for coarse-graining)                      | Model                         | 13.9.2 (using --sirah flag)       |
| AMBAT (Amber Membrane Builder and Analysis Tool)             | Software/Tool                 | 13.10                              |
| LEaP (tleap, xleap)                                          | Software/Tool                 | 14.                                |
| Object File Format (OFF)                                     | File Format                   | 14.1                               |
| PREP input files                                             | File Format                   | 14.1                               |
| PARM format parameter sets                                   | File Format                   | 14.1                               |
| loadPdb                                                      | LEaP Command                  | 14.1, 14.4.1, 14.6.29              |
| UNIT (LEaP object)                                           | Data Structure                | 14.2.1, 14.2.3                     |
| RESIDUE (LEaP object)                                        | Data Structure                | 14.2.1, 14.2.3                     |
| ATOM (LEaP object)                                           | Data Structure                | 14.2.1, 14.2.3                     |
| PARMSET (LEaP object)                                        | Data Structure                | 14.2.1, 14.2.3                     |
| leaprc file                                                  | Configuration File            | 14.3, 14.6.7, 14.6.8               |
| Universe Editor (xleap)                                      | Software Component            | 14.3.1                             |
| Unit Editor (xleap)                                          | Software Component            | 14.3.2                             |
| Atom Properties Editor (xleap)                               | Software Component            | 14.3.3                             |
| Parmset Editor (xleap)                                       | Software Component            | 14.3.4                             |
| loadOff                                                      | LEaP Command                  | 14.4.1, 14.6.27                    |
| loadAmberPrep                                                | LEaP Command                  | 14.4.1, 14.6.26                    |
| zMatrix command (LEaP)                                       | LEaP Command                  | 14.4.1, 14.6.49                    |
| loadMol2, loadMol3                                           | LEaP Command                  | 14.4.1, 14.6.28                    |
| addPdbAtomMap                                                | LEaP Command                  | 14.4.1, 14.6.8                     |
| addPdbResMap                                                 | LEaP Command                  | 14.4.1, 14.6.9                     |
| saveAmberParm                                                | LEaP Command                  | 14.4.1, 14.6.35                    |
| saveAmberParmPol                                             | LEaP Command                  | 14.4.1                             |
| NALA, CALA (N-terminal, C-terminal Alanine)                  | Residue Convention            | 14.4.2                             |
| DAN (neutral deoxyadenosine nucleoside)                      | Residue Convention            | 14.4.3                             |
| add (LEaP command)                                           | LEaP Command                  | 14.6.1                             |
| addAtomTypes                                                 | LEaP Command                  | 14.2.3, 14.6.2                     |
| addC4Pairwise                                                | LEaP Command                  | 14.6.3                             |
| addC4Type                                                    | LEaP Command                  | 14.6.4                             |
| addIons, addIons2, addIonsRand                               | LEaP Command                  | 14.6.5, 14.6.6                     |
| addPath                                                      | LEaP Command                  | 14.6.7                             |
| alias (LEaP command)                                         | LEaP Command                  | 14.6.10                            |
| bond (LEaP command)                                          | LEaP Command                  | 14.6.11                            |
| bondByDistance                                               | LEaP Command                  | 14.6.12                            |
| check (LEaP command)                                         | LEaP Command                  | 14.6.13                            |
| combine (LEaP command)                                       | LEaP Command                  | 14.6.14                            |
| copy (LEaP command)                                          | LEaP Command                  | 14.6.15                            |
| createAtom                                                   | LEaP Command                  | 14.6.16                            |
| createResidue                                                | LEaP Command                  | 14.6.17                            |
| createUnit                                                   | LEaP Command                  | 14.6.18                            |
| deleteBond                                                   | LEaP Command                  | 14.6.19                            |
| desc (LEaP command)                                          | LEaP Command                  | 14.2.2, 14.6.20                    |
| groupSelectedAtoms                                           | LEaP Command                  | 14.6.21                            |
| help (LEaP command)                                          | LEaP Command                  | 14.6.22                            |
| impose (LEaP command)                                        | LEaP Command                  | 14.6.23                            |
| list (LEaP command)                                          | LEaP Command                  | 14.2.2, 14.6.24                    |
| loadAmberParams                                              | LEaP Command                  | 14.6.25                            |
| logFile (LEaP command)                                       | LEaP Command                  | 14.3, 14.6.31                      |
| measureGeom                                                  | LEaP Command                  | 14.6.32                            |
| quit (LEaP command)                                          | LEaP Command                  | 14.6.33                            |
| remove (LEaP command)                                        | LEaP Command                  | 14.6.34                            |
| saveMol2                                                     | LEaP Command                  | 14.6.36                            |
| saveOff                                                      | LEaP Command                  | 14.6.37                            |
| savePdb                                                      | LEaP Command                  | 14.6.38                            |
| sequence (LEaP command)                                      | LEaP Command                  | 14.6.39                            |
| set (LEaP command)                                           | LEaP Command                  | 14.2.3, 14.6.40                    |
| OldPrmtopFormat (LEaP set option)                            | LEaP Option                   | 14.6.40                            |
| Dielectric (LEaP set option)                                 | LEaP Option                   | 14.6.40                            |
| PdbWriteCharges (LEaP set option)                            | LEaP Option                   | 14.6.40                            |
| PBRadii (LEaP set option)                                    | LEaP Option                   | 14.6.40                            |
| nocenter (LEaP set option)                                   | LEaP Option                   | 14.6.40                            |
| reorder residues (LEaP set option)                           | LEaP Option                   | 14.6.40                            |
| setBox (LEaP command)                                        | LEaP Command                  | 14.6.41                            |
| solvateBox, solvateOct                                       | LEaP Command                  | 14.6.42                            |
| solvateCap                                                   | LEaP Command                  | 14.6.43                            |
| solvateShell                                                 | LEaP Command                  | 14.6.44                            |
| source (LEaP command)                                        | LEaP Command                  | 14.3, 14.6.45                      |
| transform (LEaP command)                                     | LEaP Command                  | 14.6.46                            |
| translate (LEaP command)                                     | LEaP Command                  | 14.6.47                            |
| verbosity (LEaP command)                                     | LEaP Command                  | 14.3, 14.6.48                      |
| GLYCAM-Web                                                   | Web Tool                      | 14.7                               |
| DMPC (1,2-dimyristoyl-sn-glycero-3-phosphocholine)           | Chemical Entity               | 14.7.2                             |
| Solvate.sh                                                   | Script                        | 14.7.4                             |
| Amber parameter files (*.dat, frcmod.*)                      | File Format                   | 15.                                |
| ParmEd                                                       | Software/Tool                 | 15., 15.2                          |
| parameter-topology file (parmtop)                            | File Format                   | 15.1.2                             |
| coordinate file (*.crd, PDB)                                 | File Format                   | 15.1.2 (footnote)                  |
| Lennard-Jones potential (6-12)                               | Model/Equation                | 15.1.7                             |
| Lorentz/Berthelot mixing rules                               | Rule/Convention               | 15.1.7                             |
| AM1-BBC method (charge calculation)                          | Method                        | 15.1.7                             |
| addAtomicNumber (ParmEd command)                             | ParmEd Command                | 15.2.2.1                           |
| addDihedral (ParmEd command)                                 | ParmEd Command                | 15.2.2.2                           |
| addExclusions (ParmEd command)                               | ParmEd Command                | 15.2.2.3                           |
| addLJType (ParmEd command)                                   | ParmEd Command                | 15.2.2.4                           |
| addPDB (ParmEd command)                                      | ParmEd Command                | 15.2.2.5                           |
| add12_6_4 (ParmEd command)                                   | ParmEd Command                | 15.2.2.6                           |
| chamber (ParmEd command)                                     | ParmEd Command                | 15.2.2.8                           |
| CHARMM force field (via chamber)                             | Model                         | 15.2.2.8                           |
| Urey-Bradley term                                            | Model Term                    | 15.2.2.8                           |
| CMAP (Correction Map)                                        | Model Term                    | 15.2.2.8                           |
| CTITLE, FORCE_FIELD_TYPE, CHARMM_UREY_BRADLEY_COUNT etc. (ParmEd %FLAGs for CHARMM) | File Format Element           | 15.2.2.8                           |
| change (ParmEd command)                                      | ParmEd Command                | 15.2.2.9                           |
| changeLJPair (ParmEd command)                                | ParmEd Command                | 15.2.2.10                          |
| changeLJ14Pair (ParmEd command)                              | ParmEd Command                | 15.2.2.11                          |
| changeLJSingleType (ParmEd command)                          | ParmEd Command                | 15.2.2.12                          |
| changeProtState (ParmEd command)                             | ParmEd Command                | 15.2.2.13                          |
| changeRedoxState (ParmEd command)                            | ParmEd Command                | 15.2.2.14                          |
| changeRadii (ParmEd command)                                 | ParmEd Command                | 15.2.2.15                          |
| checkValidity (ParmEd command)                               | ParmEd Command                | 15.2.2.16                          |
| defineSolvent (ParmEd command)                               | ParmEd Command                | 15.2.2.17                          |
| deleteBond (ParmEd command)                                  | ParmEd Command                | 15.2.2.18                          |
| deleteDihedral (ParmEd command)                              | ParmEd Command                | 15.2.2.19                          |
| deletePDB (ParmEd command)                                   | ParmEd Command                | 15.2.2.20                          |
| energy (ParmEd command)                                      | ParmEd Command                | 15.2.2.21                          |
| OpenMM                                                       | Software/API                  | 15.2.2.21, 15.2.2.33, 15.2.2.35    |
| go (ParmEd command)                                          | ParmEd Command                | 15.2.2.22                          |
| gromber (ParmEd command)                                     | ParmEd Command                | 15.2.2.23                          |
| HMassRepartition (ParmEd command)                            | ParmEd Command                | 15.2.2.24                          |
| history (ParmEd command)                                     | ParmEd Command                | 15.2.2.26                          |
| interpolate (ParmEd command)                                 | ParmEd Command                | 15.2.2.27                          |
| listParms (ParmEd command)                                   | ParmEd Command                | 15.2.2.28                          |
| lmod (ParmEd command)                                        | ParmEd Command                | 15.2.2.29                          |
| loadCoordinates (ParmEd command)                             | ParmEd Command                | 15.2.2.30                          |
| loadRestrt (ParmEd command)                                  | ParmEd Command                | 15.2.2.21, 15.2.2.31, 15.2.2.38, 15.2.2.40 |
| ls (ParmEd command)                                          | ParmEd Command                | 15.2.2.32                          |
| minimize (ParmEd command)                                    | ParmEd Command                | 15.2.2.33                          |
| netCharge (ParmEd command)                                   | ParmEd Command                | 15.2.2.34                          |
| outCIF (ParmEd command)                                      | ParmEd Command                | 15.2.2.36                          |
| PDBx/mmCIF file format                                       | File Format                   | 15.2.2.30, 15.2.2.36               |
| outparm (ParmEd command)                                     | ParmEd Command                | 15.2.2.37                          |
| outPDB (ParmEd command)                                      | ParmEd Command                | 15.2.2.38                          |
| parm (ParmEd command)                                        | ParmEd Command                | 15.2.2.39                          |
| parmout (ParmEd command)                                     | ParmEd Command                | 15.2.2.40                          |
| printAngles (ParmEd command)                                 | ParmEd Command                | 15.2.2.41                          |
| printBonds (ParmEd command)                                  | ParmEd Command                | 15.2.2.42                          |
| printDetails (ParmEd command)                                | ParmEd Command                | 15.2.2.43                          |
| printDihedrals (ParmEd command)                              | ParmEd Command                | 15.2.2.44                          |
| printFlags (ParmEd command)                                  | ParmEd Command                | 15.2.2.45                          |
| printInfo (ParmEd command)                                   | ParmEd Command                | 15.2.2.46                          |
| printLJMatrix (ParmEd command)                               | ParmEd Command                | 15.2.2.47                          |
| printLJTypes (ParmEd command)                                | ParmEd Command                | 15.2.2.48                          |
| printPointers (ParmEd command)                               | ParmEd Command                | 15.2.2.49                          |
| quit (ParmEd command)                                        | ParmEd Command                | 15.2.2.50                          |
| scale (ParmEd command)                                       | ParmEd Command                | 15.2.2.51                          |
| scee (ParmEd command)                                        | ParmEd Command                | 15.2.2.52                          |
| scnb (ParmEd command)                                        | ParmEd Command                | 15.2.2.53                          |
| setAngle (ParmEd command)                                    | ParmEd Command                | 15.2.2.54                          |
| setBond (ParmEd command)                                     | ParmEd Command                | 15.2.2.18, 15.2.2.55               |
| setMolecules (ParmEd command)                                | ParmEd Command                | 15.2.2.56                          |
| setOverwrite (ParmEd command)                                | ParmEd Command                | 15.2.2.57                          |
| source (ParmEd command)                                      | ParmEd Command                | 15.2.2.58                          |
| strip (ParmEd command)                                       | ParmEd Command                | 15.2.2.37, 15.2.2.59               |
| summary (ParmEd command)                                     | ParmEd Command                | 15.2.2.60                          |
| tiMerge (ParmEd command)                                     | ParmEd Command                | 15.2.2.61                          |
| writeFrcmod (ParmEd command)                                 | ParmEd Command                | 15.2.2.62                          |
| writeOFF (ParmEd command)                                    | ParmEd Command                | 15.2.2.63                          |
| Interactive Python Shell (ParmEd)                            | Software Feature              | 15.2.5.1                           |
| Antechamber                                                  | Software Suite                | 16.                                |
| GAFF (General Amber Force Field)                             | Model                         | 16.                                |
| gaff.dat, gaff2.dat                                          | File/Parameter Set            | 16.                                |
| pyMSMT (Python Metal Site Modeling Toolbox)                  | Software Suite                | 16., 19.                           |
| HF/6-31G* RESP (charge method)                               | Method                        | 16.                                |
| AM1-BCC (charge method)                                      | Method                        | 16., 16.1.2, 16.3.1, 16.3.2        |
| ABCG2 (charge method)                                        | Method                        | 16.                                |
| PARMSCAN                                                     | Software Package              | 16.                                |
| antechamber (program)                                        | Software                      | 16.1.1                             |
| sqm (program, for antechamber)                               | Software                      | 16.1.1, 16.3.1                     |
| mopac (program, for antechamber)                             | Software                      | 16.1.1                             |
| divcon (program, for antechamber)                            | Software                      | 16.1.1                             |
| atomtype (program)                                           | Software                      | 16.1.1, 16.3.1                     |
| bondtype (program)                                           | Software                      | 16.1.1, 16.3.3, 16.4.1             |
| espgen (program)                                             | Software                      | 16.1.1, 16.3.5, 19.2.6, 20.2       |
| respgen (program)                                            | Software                      | 16.1.1, 16.3.6                     |
| prepgen (program)                                            | Software                      | 16.1.1, 16.3.4, 16.4.3             |
| parmchk2 (program)                                           | Software                      | 16.1.1, 16.1.2                     |
| PARMCHK.DAT                                                  | File/Parameter Set            | 16.1.2                             |
| ATCOR.DAT                                                    | File/Parameter Set            | 16.1.2                             |
| acdoctor (program)                                           | Software                      | 16.3.1, 16.4.1                     |
| parmcal (program)                                            | Software                      | 16.4.2                             |
| residuegen (program)                                         | Software                      | 16.4.3                             |
| match (program)                                              | Software                      | 16.4.4                             |
| match_atomname (program)                                     | Software                      | 16.4.5                             |
| paramfit                                                     | Software                      | 17.                                |
| Job Control File (paramfit)                                  | File Format                   | 17.2                               |
| Genetic Algorithm (paramfit)                                 | Algorithm                     | 17.2.4, 17.2.5                     |
| Simplex Algorithm (paramfit)                                 | Algorithm                     | 17.2.4, 17.2.5                     |
| mdgx                                                         | Software                      | 18.                                |
| IPolQ (Implicitly Polarized Charge model)                    | Model                         | 18., 18.1, 18.4                    |
| RESP (Restrained ElectroStatic Potential)                    | Method/Algorithm              | 18.3, 20.                          |
| Virtual Site Support (mdgx)                                  | Software Feature              | 18.5                               |
| CMAP surfaces (fitting in mdgx)                              | Model Term/Parameter          | 18.6                               |
| Configuration Sampling (mdgx)                                | Method                        | 18.7                               |
| Peptide multi-simulator (mdgx GPU)                           | Software Feature              | 18.8                               |
| RATTLE (constraint algorithm)                                | Algorithm                     | 18.8                               |
| I/r-RESPA (multi-time stepping)                              | Algorithm                     | 18.8                               |
| MCPB.py (Metal Center Parameter Builder)                     | Software                      | 19.1, 19.2.1                       |
| IPMach.py (Ion Parameterization Machine)                     | Software                      | 19.1, 19.2.2                       |
| PdbSearcher.py                                               | Software                      | 19.1, 19.2.3                       |
| OptC4.py                                                     | Software                      | 19.1, 19.2.4                       |
| CartHess2FC.py                                               | Software                      | 19.1, 19.2.5                       |
| ProScrs.py (Protein Scissors)                                | Software                      | 19.1, 19.2.7                       |
| car_to_files.py                                              | Software                      | 19.1, 19.2.8                       |
| mol2rtf.py                                                   | Software                      | 19.1, 19.2.9                       |
| metalpdb2mol2.py                                             | Software                      | 19.1, 19.2.10                      |
| py_resp.py                                                   | Software                      | 20.                                |
| RESP-ind model                                               | Model                         | 20., 20.2, 20.3.1, 20.4.5          |
| RESP-perm model                                              | Model                         | 20., 20.2, 20.3.1, 20.4.5          |
| RESP-perm-v model                                            | Model                         | 20., 20.3.1                        |
| pyresp_gen.py                                                | Software                      | 20.1                               |
| pGM damping function                                         | Function/Model Feature        | 20.1.1, 20.2                       |
| PyPE_RESP                                                    | Software/Pipeline             | 20.4                               |
| RDKit                                                        | Software Package              | 20.4.2                             |
| SMARTS patterns                                              | Syntax/Convention             | 20.4.2                             |
| UnitCell (program for crystal simulations)                   | Software                      | 21.1                               |
| PropPDB (program for crystal simulations)                    | Software                      | 21.2                               |
| AddToBox (program for crystal simulations)                   | Software                      | 21.3                               |
| ChBox (program for crystal simulations)                      | Software                      | 21.4                               |
| sander (simulation engine)                                   | Software                      | 22.                                |
| Particle-mesh Ewald (PME)                                    | Algorithm/Method              | 22.1                               |
| NVE ensemble                                                 | Simulation Condition          | 22.1                               |
| NVT ensemble                                                 | Simulation Condition          | 22.6.2 (nmropt, implicit)          |
| Generalized Born/surface area model (in sander)              | Model                         | 22.1                               |
| NOESY intensities                                            | Data Type                     | 22.1                               |
| Residual dipolar couplings                                   | Data Type                     | 22.1                               |
| Scalar coupling constants                                    | Data Type                     | 22.1                               |
| Proton chemical shifts                                       | Data Type                     | 22.1                               |
| Replica exchange calculations                                | Method                        | 22.1                               |
| Thermodynamic integration (TI, in sander)                    | Method                        | 22.1                               |
| Umbrella sampling (in sander)                                | Method                        | 22.1                               |
| Empirical valence bond (EVB) scheme                          | Method                        | 22.1                               |
| QMMM Calculations (in sander)                                | Method                        | 22.1                               |
| Path-integral molecular dynamics (PIMD)                      | Method                        | 22.1                               |
| mdin, mdout, mdinfo, prmtop, inpcrd, refc, mtmd, mdcrd, inptraj, mdvel, mdfrc, mden, restrt, inpdip, rstdip, cpin, cprestrt, cpout, cein, cerestrt, ceout, evbin (sander file types) | File Type                     | 22.2                               |
| Namelist Input Syntax (sander)                               | Syntax/Convention             | 22.4                               |
| &cntrl, &pb, &ewald, &qmmm, &amoeba, &debugf (sander namelists) | Namelist                      | 22.5                               |
| &wt namelist (sander varying conditions)                     | Namelist                      | 22.5                               |
| Frozen/restrained atoms (sander)                             | Simulation Feature            | 22.6.4                             |
| Belly type dynamics                                          | Method                        | 22.6.4                             |
| adQM/MM (adaptive QM/MM)                                     | Method                        | 11.8                               |
| A region (adQM/MM)                                           | Region/Concept                | 11.8.2.1, 11.8.2.2                 |
| T region (adQM/MM)                                           | Region/Concept                | 11.8.2.1, 11.8.2.2                 |
| multisander                                                  | Software Feature              | 11.8.2                             |
| groupfile (for multisander in adQM/MM)                       | File Format                   | 11.8.2                             |
| &vsolv namelist (adQM/MM)                                    | Namelist                      | 11.8.2.2                           |
| &adqmmm namelist (adQM/MM)                                   | Namelist                      | 11.8.2.2                           |
| Adaptive buffered force-mixing QM/MM (abfQM/MM)              | Method                        | 11.9                               |
| extended system (abfQM/MM)                                   | System Definition             | 11.9.4.1                           |
| reduced system (abfQM/MM)                                    | System Definition             | 11.9.4.1                           |
| core atoms (abfQM/MM)                                        | Atom Type (abfQM/MM)          | 11.9.4.2                           |
| qm atoms (abfQM/MM)                                          | Atom Type (abfQM/MM)          | 11.9.4.2                           |
| buffer atoms (abfQM/MM)                                      | Atom Type (abfQM/MM)          | 11.9.4.2                           |
| mm atoms (abfQM/MM)                                          | Atom Type (abfQM/MM)          | 11.9.4.2                           |
| &qmmm namelist (for abfQM/MM)                                | Namelist                      | 11.9.5                             |
| ambmask format                                               | Data Format/Syntax            | 11.9.5.1                           |
| Adaptive thermostats (Nose-Hoover, Langevin for abfQM/MM)    | Algorithm/Method              | 11.9.5.2                           |
| SEBOMD (SemiEmpirical Born-Oppenheimer Molecular Dynamics)   | Method/Software Feature       | 11.10                              |
| DivCon                                                       | Software                      | 11.10                              |
| &sebomd namelist                                             | Namelist                      | 11.10.2, 11.10.3                   |
| PIF2, PIF3 (PM3 corrections for SEBOMD)                      | Correction/Model              | 11.10.3 (modif)                    |
| MAIS1, MAIS2 (PM3 corrections for SEBOMD)                    | Correction/Model              | 11.10.3 (modif)                    |
| ReaxFF/AMBER                                                 | Method/Software Feature       | 11.11                              |
| PuReMD (Reactive Molecular Dynamics)                         | Software                      | 11.11                              |
| ReaxFF force fields                                          | Model                         | 11.11.2                            |
| ffield.reaxff                                                | File Format/Default Name      | 11.11.4                            |
| &reaxff namelist                                             | Namelist                      | 11.11.4, 11.11.5                   |
| QEq (Charge equilibration for ReaxFF)                        | Method                        | 11.11.5 (charge_method)            |
| EE (Electronegativity equalization for ReaxFF)               | Method                        | 11.11.5 (charge_method)            |
| ACKS2 (Atom-condensed Kohn-Sham approximated to second order for ReaxFF) | Method                        | 11.11.5 (charge_method)            |
| reaxff-puremd control file                                   | File Format                   | 11.11.6                            |
| External library interface (Amber)                           | Software Feature              | 12.                                |
| MBX library                                                  | Software/Library              | 12.                                |
| KMMD library (Machine Learning Molecular Dynamics)           | Software/Library              | 12.                                |
| &extpot namelist                                             | Namelist                      | 12.2                               |
| PDB Files (Preparation)                                      | Process                       | 13.                                |
| pdb4amber                                                    | Software/Tool                 | 13.                                |
| reduce (program for adding hydrogens)                        | Software/Tool                 | 13., 13.5                          |
| TER record (PDB)                                             | Data Format Element           | 13.1, 13.2, 13.3                   |
| HIE, HID, HIP (Histidine protonation states)                 | Chemical State/Convention     | 13.2                               |
| CYX (Cysteine in disulfide bridge)                           | Chemical State/Convention     | 13.2                               |
| ASH, GLH, LYN (Neutral Asp, Glu, Lys)                        | Chemical State/Convention     | 13.2                               |
| ACE, NHE, NME (Terminal cap residues)                        | Chemical Entity/Convention    | 13.2, 13.3                         |
| AMBER masks                                                  | Syntax/Convention             | 13.3                               |
| NAB atom expressions                                         | Syntax/Convention             | 13.3                               |
| pytleap                                                      | Software/Script               | 13.3                               |
| pdb2pqr                                                      | Software/Tool                 | 13.6                               |
| propka3                                                      | Software (used by pdb2pqr)    | 13.6                               |
| gwh (guess water hydrogens)                                  | Software/Tool                 | 13.7                               |
| packmol-memgen                                               | Software/Tool                 | 13.9                               |
| Memembed                                                     | Software (used by packmol-memgen) | 13.9                               |
| pdbremix                                                     | Software (used by packmol-memgen) | 13.9                               |
| Packmol                                                      | Software (used by packmol-memgen) | 13.9                               |
| charmmlipid2amber.py                                         | Software/Script               | 13.9                               |
| OPM/PPM3 orientation program                                 | Software                      | 13.9                               |
| immers                                                       | Software                      | 13.9                               |
| Lipid ext (Lipid extension force field)                      | Model                         | 13.9.1                             |
| lysophospholipids                                            | Chemical Entity Class         | 13.9.1                             |
| phosphatidylinositol                                         | Chemical Entity Class         | 13.9.1                             |
| cardiolipins                                                 | Chemical Entity Class         | 13.9.1                             |
| sterols (ergosterol, campesterol, sitosterol, stigmasterol)  | Chemical Entity Class         | 13.9.1                             |
| memgen.parm file                                             | File Format                   | 13.9.2                             |
| SIRAH force field (for coarse-graining in packmol-memgen)    | Model                         | 13.9.2 (using --sirah flag)       |
| AMBAT (Amber Membrane Builder and Analysis Tool)             | Software/Tool                 | 13.10                              |
| LEaP (tleap, xleap)                                          | Software/Tool                 | 14.                                |
| Object File Format (OFF)                                     | File Format                   | 14.1                               |
| PREP input files                                             | File Format                   | 14.1                               |
| PARM format parameter sets                                   | File Format                   | 14.1                               |
| loadPdb                                                      | LEaP Command                  | 14.1, 14.4.1, 14.6.29              |
| UNIT (LEaP object)                                           | Data Structure                | 14.2.1, 14.2.3                     |
| RESIDUE (LEaP object)                                        | Data Structure                | 14.2.1, 14.2.3                     |
| ATOM (LEaP object)                                           | Data Structure                | 14.2.1, 14.2.3                     |
| PARMSET (LEaP object)                                        | Data Structure                | 14.2.1, 14.2.3                     |
| leaprc file                                                  | Configuration File            | 14.3, 14.6.7, 14.6.8               |
| Universe Editor (xleap)                                      | Software Component            | 14.3.1                             |
| Unit Editor (xleap)                                          | Software Component            | 14.3.2                             |
| Atom Properties Editor (xleap)                               | Software Component            | 14.3.3                             |
| Parmset Editor (xleap)                                       | Software Component            | 14.3.4                             |
| loadOff                                                      | LEaP Command                  | 14.4.1, 14.6.27                    |
| loadAmberPrep                                                | LEaP Command                  | 14.4.1, 14.6.26                    |
| zMatrix command (LEaP)                                       | LEaP Command                  | 14.4.1, 14.6.49                    |
| loadMol2, loadMol3                                           | LEaP Command                  | 14.4.1, 14.6.28                    |
| addPdbAtomMap                                                | LEaP Command                  | 14.4.1, 14.6.8                     |
| addPdbResMap                                                 | LEaP Command                  | 14.4.1, 14.6.9                     |
| saveAmberParm                                                | LEaP Command                  | 14.4.1, 14.6.35                    |
| saveAmberParmPol                                             | LEaP Command                  | 14.4.1                             |
| NALA, CALA (N-terminal, C-terminal Alanine)                  | Residue Convention            | 14.4.2                             |
| DAN (neutral deoxyadenosine nucleoside)                      | Residue Convention            | 14.4.3                             |
| add (LEaP command)                                           | LEaP Command                  | 14.6.1                             |
| addAtomTypes                                                 | LEaP Command                  | 14.2.3, 14.6.2                     |
| addC4Pairwise                                                | LEaP Command                  | 14.6.3                             |
| addC4Type                                                    | LEaP Command                  | 14.6.4                             |
| addIons, addIons2, addIonsRand                               | LEaP Command                  | 14.6.5, 14.6.6                     |
| addPath                                                      | LEaP Command                  | 14.6.7                             |
| alias (LEaP command)                                         | LEaP Command                  | 14.6.10                            |
| bond (LEaP command)                                          | LEaP Command                  | 14.6.11                            |
| bondByDistance                                               | LEaP Command                  | 14.6.12                            |
| check (LEaP command)                                         | LEaP Command                  | 14.6.13                            |
| combine (LEaP command)                                       | LEaP Command                  | 14.6.14                            |
| copy (LEaP command)                                          | LEaP Command                  | 14.6.15                            |
| createAtom                                                   | LEaP Command                  | 14.6.16                            |
| createResidue                                                | LEaP Command                  | 14.6.17                            |
| createUnit                                                   | LEaP Command                  | 14.6.18                            |
| deleteBond                                                   | LEaP Command                  | 14.6.19                            |
| desc (LEaP command)                                          | LEaP Command                  | 14.2.2, 14.6.20                    |
| groupSelectedAtoms                                           | LEaP Command                  | 14.6.21                            |
| help (LEaP command)                                          | LEaP Command                  | 14.6.22                            |
| impose (LEaP command)                                        | LEaP Command                  | 14.6.23                            |
| list (LEaP command)                                          | LEaP Command                  | 14.2.2, 14.6.24                    |
| loadAmberParams                                              | LEaP Command                  | 14.6.25                            |
| logFile (LEaP command)                                       | LEaP Command                  | 14.3, 14.6.31                      |
| measureGeom                                                  | LEaP Command                  | 14.6.32                            |
| quit (LEaP command)                                          | LEaP Command                  | 14.6.33                            |
| remove (LEaP command)                                        | LEaP Command                  | 14.6.34                            |
| saveMol2                                                     | LEaP Command                  | 14.6.36                            |
| saveOff                                                      | LEaP Command                  | 14.6.37                            |
| savePdb                                                      | LEaP Command                  | 14.6.38                            |
| sequence (LEaP command)                                      | LEaP Command                  | 14.6.39                            |
| set (LEaP command)                                           | LEaP Command                  | 14.2.3, 14.6.40                    |
| OldPrmtopFormat (LEaP set option)                            | LEaP Option                   | 14.6.40                            |
| Dielectric (LEaP set option)                                 | LEaP Option                   | 14.6.40                            |
| PdbWriteCharges (LEaP set option)                            | LEaP Option                   | 14.6.40                            |
| PBRadii (LEaP set option)                                    | LEaP Option                   | 14.6.40                            |
| nocenter (LEaP set option)                                   | LEaP Option                   | 14.6.40                            |
| reorder residues (LEaP set option)                           | LEaP Option                   | 14.6.40                            |
| setBox (LEaP command)                                        | LEaP Command                  | 14.6.41                            |
| solvateBox, solvateOct                                       | LEaP Command                  | 14.6.42                            |
| solvateCap                                                   | LEaP Command                  | 14.6.43                            |
| solvateShell                                                 | LEaP Command                  | 14.6.44                            |
| source (LEaP command)                                        | LEaP Command                  | 14.3, 14.6.45                      |
| transform (LEaP command)                                     | LEaP Command                  | 14.6.46                            |
| translate (LEaP command)                                     | LEaP Command                  | 14.6.47                            |
| verbosity (LEaP command)                                     | LEaP Command                  | 14.3, 14.6.48                      |
| GLYCAM-Web                                                   | Web Tool                      | 14.7                               |
| DMPC (1,2-dimyristoyl-sn-glycero-3-phosphocholine)           | Chemical Entity               | 14.7.2                             |
| Solvate.sh                                                   | Script                        | 14.7.4                             |
| Amber parameter files (*.dat, frcmod.*)                      | File Format                   | 15.                                |
| ParmEd                                                       | Software/Tool                 | 15., 15.2                          |
| parameter-topology file (parmtop)                            | File Format                   | 15.1.2                             |
| coordinate file (*.crd, PDB)                                 | File Format                   | 15.1.2 (footnote)                  |
| Lennard-Jones potential (6-12)                               | Model/Equation                | 15.1.7                             |
| Lorentz/Berthelot mixing rules                               | Rule/Convention               | 15.1.7                             |
| AM1-BBC method (charge calculation)                          | Method                        | 15.1.7                             |
| addAtomicNumber (ParmEd command)                             | ParmEd Command                | 15.2.2.1                           |
| addDihedral (ParmEd command)                                 | ParmEd Command                | 15.2.2.2                           |
| addExclusions (ParmEd command)                               | ParmEd Command                | 15.2.2.3                           |
| addLJType (ParmEd command)                                   | ParmEd Command                | 15.2.2.4                           |
| addPDB (ParmEd command)                                      | ParmEd Command                | 15.2.2.5                           |
| add12_6_4 (ParmEd command)                                   | ParmEd Command                | 15.2.2.6                           |
| chamber (ParmEd command)                                     | ParmEd Command                | 15.2.2.8                           |
| CHARMM force field (via chamber)                             | Model                         | 15.2.2.8                           |
| Urey-Bradley term                                            | Model Term                    | 15.2.2.8                           |
| CMAP (Correction Map)                                        | Model Term                    | 15.2.2.8                           |
| CTITLE, FORCE_FIELD_TYPE, CHARMM_UREY_BRADLEY_COUNT etc. (ParmEd %FLAGs for CHARMM) | File Format Element           | 15.2.2.8                           |
| change (ParmEd command)                                      | ParmEd Command                | 15.2.2.9                           |
| changeLJPair (ParmEd command)                                | ParmEd Command                | 15.2.2.10                          |
| changeLJ14Pair (ParmEd command)                              | ParmEd Command                | 15.2.2.11                          |
| changeLJSingleType (ParmEd command)                          | ParmEd Command                | 15.2.2.12                          |
| changeProtState (ParmEd command)                             | ParmEd Command                | 15.2.2.13                          |
| changeRedoxState (ParmEd command)                            | ParmEd Command                | 15.2.2.14                          |
| changeRadii (ParmEd command)                                 | ParmEd Command                | 15.2.2.15                          |
| checkValidity (ParmEd command)                               | ParmEd Command                | 15.2.2.16                          |
| defineSolvent (ParmEd command)                               | ParmEd Command                | 15.2.2.17                          |
| deleteBond (ParmEd command)                                  | ParmEd Command                | 15.2.2.18                          |
| deleteDihedral (ParmEd command)                              | ParmEd Command                | 15.2.2.19                          |
| deletePDB (ParmEd command)                                   | ParmEd Command                | 15.2.2.20                          |
| energy (ParmEd command)                                      | ParmEd Command                | 15.2.2.21                          |
| OpenMM                                                       | Software/API                  | 15.2.2.21, 15.2.2.33, 15.2.2.35    |
| go (ParmEd command)                                          | ParmEd Command                | 15.2.2.22                          |
| gromber (ParmEd command)                                     | ParmEd Command                | 15.2.2.23                          |
| HMassRepartition (ParmEd command)                            | ParmEd Command                | 15.2.2.24                          |
| history (ParmEd command)                                     | ParmEd Command                | 15.2.2.26                          |
| interpolate (ParmEd command)                                 | ParmEd Command                | 15.2.2.27                          |
| listParms (ParmEd command)                                   | ParmEd Command                | 15.2.2.28                          |
| lmod (ParmEd command)                                        | ParmEd Command                | 15.2.2.29                          |
| loadCoordinates (ParmEd command)                             | ParmEd Command                | 15.2.2.30                          |
| loadRestrt (ParmEd command)                                  | ParmEd Command                | 15.2.2.21, 15.2.2.31, 15.2.2.38, 15.2.2.40 |
| ls (ParmEd command)                                          | ParmEd Command                | 15.2.2.32                          |
| minimize (ParmEd command)                                    | ParmEd Command                | 15.2.2.33                          |
| netCharge (ParmEd command)                                   | ParmEd Command                | 15.2.2.34                          |
| outCIF (ParmEd command)                                      | ParmEd Command                | 15.2.2.36                          |
| PDBx/mmCIF file format                                       | File Format                   | 15.2.2.30, 15.2.2.36               |
| outparm (ParmEd command)                                     | ParmEd Command                | 15.2.2.37                          |
| outPDB (ParmEd command)                                      | ParmEd Command                | 15.2.2.38                          |
| parm (ParmEd command)                                        | ParmEd Command                | 15.2.2.39                          |
| parmout (ParmEd command)                                     | ParmEd Command                | 15.2.2.40                          |
| printAngles (ParmEd command)                                 | ParmEd Command                | 15.2.2.41                          |
| printBonds (ParmEd command)                                  | ParmEd Command                | 15.2.2.42                          |
| printDetails (ParmEd command)                                | ParmEd Command                | 15.2.2.43                          |
| printDihedrals (ParmEd command)                              | ParmEd Command                | 15.2.2.44                          |
| printFlags (ParmEd command)                                  | ParmEd Command                | 15.2.2.45                          |
| printInfo (ParmEd command)                                   | ParmEd Command                | 15.2.2.46                          |
| printLJMatrix (ParmEd command)                               | ParmEd Command                | 15.2.2.47                          |
| printLJTypes (ParmEd command)                                | ParmEd Command                | 15.2.2.48                          |
| printPointers (ParmEd command)                               | ParmEd Command                | 15.2.2.49                          |
| quit (ParmEd command)                                        | ParmEd Command                | 15.2.2.50                          |
| scale (ParmEd command)                                       | ParmEd Command                | 15.2.2.51                          |
| scee (ParmEd command)                                        | ParmEd Command                | 15.2.2.52                          |
| scnb (ParmEd command)                                        | ParmEd Command                | 15.2.2.53                          |
| setAngle (ParmEd command)                                    | ParmEd Command                | 15.2.2.54                          |
| setBond (ParmEd command)                                     | ParmEd Command                | 15.2.2.18, 15.2.2.55               |
| setMolecules (ParmEd command)                                | ParmEd Command                | 15.2.2.56                          |
| setOverwrite (ParmEd command)                                | ParmEd Command                | 15.2.2.57                          |
| source (ParmEd command)                                      | ParmEd Command                | 15.2.2.58                          |
| strip (ParmEd command)                                       | ParmEd Command                | 15.2.2.37, 15.2.2.59               |
| summary (ParmEd command)                                     | ParmEd Command                | 15.2.2.60                          |
| tiMerge (ParmEd command)                                     | ParmEd Command                | 15.2.2.61                          |
| writeFrcmod (ParmEd command)                                 | ParmEd Command                | 15.2.2.62                          |
| writeOFF (ParmEd command)                                    | ParmEd Command                | 15.2.2.63                          |
| Interactive Python Shell (ParmEd)                            | Software Feature              | 15.2.5.1                           |
| Antechamber                                                  | Software Suite                | 16.                                |
| GAFF (General Amber Force Field)                             | Model                         | 16.                                |
| gaff.dat, gaff2.dat                                          | File/Parameter Set            | 16.                                |
| pyMSMT (Python Metal Site Modeling Toolbox)                  | Software Suite                | 16., 19.                           |
| HF/6-31G* RESP (charge method)                               | Method                        | 16.                                |
| AM1-BCC (charge method)                                      | Method                        | 16., 16.1.2, 16.3.1, 16.3.2        |
| ABCG2 (charge method)                                        | Method                        | 16.                                |
| PARMSCAN                                                     | Software Package              | 16.                                |
| antechamber (program)                                        | Software                      | 16.1.1                             |
| sqm (program, for antechamber)                               | Software                      | 16.1.1, 16.3.1                     |
| mopac (program, for antechamber)                             | Software                      | 16.1.1                             |
| divcon (program, for antechamber)                            | Software                      | 16.1.1                             |
| atomtype (program)                                           | Software                      | 16.1.1, 16.3.1                     |
| bondtype (program)                                           | Software                      | 16.1.1, 16.3.3, 16.4.1             |
| espgen (program)                                             | Software                      | 16.1.1, 16.3.5, 19.2.6, 20.2       |
| respgen (program)                                            | Software                      | 16.1.1, 16.3.6                     |
| prepgen (program)                                            | Software                      | 16.1.1, 16.3.4, 16.4.3             |
| parmchk2 (program)                                           | Software                      | 16.1.1, 16.1.2                     |
| PARMCHK.DAT                                                  | File/Parameter Set            | 16.1.2                             |
| ATCOR.DAT                                                    | File/Parameter Set            | 16.1.2                             |
| acdoctor (program)                                           | Software                      | 16.3.1, 16.4.1                     |
| parmcal (program)                                            | Software                      | 16.4.2                             |
| residuegen (program)                                         | Software                      | 16.4.3                             |
| match (program)                                              | Software                      | 16.4.4                             |
| match_atomname (program)                                     | Software                      | 16.4.5                             |
| paramfit                                                     | Software                      | 17.                                |
| Job Control File (paramfit)                                  | File Format                   | 17.2                               |
| Genetic Algorithm (paramfit)                                 | Algorithm                     | 17.2.4, 17.2.5                     |
| Simplex Algorithm (paramfit)                                 | Algorithm                     | 17.2.4, 17.2.5                     |
| mdgx                                                         | Software                      | 18.                                |
| IPolQ (Implicitly Polarized Charge model)                    | Model                         | 18., 18.1, 18.4                    |
| RESP (Restrained ElectroStatic Potential)                    | Method/Algorithm              | 18.3, 20.                          |
| Virtual Site Support (mdgx)                                  | Software Feature              | 18.5                               |
| CMAP surfaces (fitting in mdgx)                              | Model Term/Parameter          | 18.6                               |
| Configuration Sampling (mdgx)                                | Method                        | 18.7                               |
| Peptide multi-simulator (mdgx GPU)                           | Software Feature              | 18.8                               |
| RATTLE (constraint algorithm)                                | Algorithm                     | 18.8                               |
| I/r-RESPA (multi-time stepping)                              | Algorithm                     | 18.8                               |
| MCPB.py (Metal Center Parameter Builder)                     | Software                      | 19.1, 19.2.1                       |
| IPMach.py (Ion Parameterization Machine)                     | Software                      | 19.1, 19.2.2                       |
| PdbSearcher.py                                               | Software                      | 19.1, 19.2.3                       |
| OptC4.py                                                     | Software                      | 19.1, 19.2.4                       |
| CartHess2FC.py                                               | Software                      | 19.1, 19.2.5                       |
| ProScrs.py (Protein Scissors)                                | Software                      | 19.1, 19.2.7                       |
| car_to_files.py                                              | Software                      | 19.1, 19.2.8                       |
| mol2rtf.py                                                   | Software                      | 19.1, 19.2.9                       |
| metalpdb2mol2.py                                             | Software                      | 19.1, 19.2.10                      |
| py_resp.py                                                   | Software                      | 20.                                |
| RESP-ind model                                               | Model                         | 20., 20.2, 20.3.1, 20.4.5          |
| RESP-perm model                                              | Model                         | 20., 20.2, 20.3.1, 20.4.5          |
| RESP-perm-v model                                            | Model                         | 20., 20.3.1                        |
| pyresp_gen.py                                                | Software                      | 20.1                               |
| pGM damping function                                         | Function/Model Feature        | 20.1.1, 20.2                       |
| PyPE_RESP                                                    | Software/Pipeline             | 20.4                               |
| RDKit                                                        | Software Package              | 20.4.2                             |
| SMARTS patterns                                              | Syntax/Convention             | 20.4.2                             |
| UnitCell (program for crystal simulations)                   | Software                      | 21.1                               |
| PropPDB (program for crystal simulations)                    | Software                      | 21.2                               |
| AddToBox (program for crystal simulations)                   | Software                      | 21.3                               |
| ChBox (program for crystal simulations)                      | Software                      | 21.4                               |
| sander (simulation engine)                                   | Software                      | 22.                                |
| Particle-mesh Ewald (PME)                                    | Algorithm/Method              | 22.1, 22.7.2                       |
| NVE ensemble                                                 | Simulation Condition          | 22.1, 22.6.7, 22.7.5, 22.8.1       |
| NVT ensemble                                                 | Simulation Condition          | 22.6.2 (nmropt, implicit), 22.8.1  |
| NPT ensemble                                                 | Simulation Condition          | 22.6.8, 22.8.1, 23.6.1, 23.6.6     |
| Generalized Born/surface area model (in sander)              | Model                         | 22.1                               |
| NOESY intensities                                            | Data Type                     | 22.1                               |
| Residual dipolar couplings                                   | Data Type                     | 22.1                               |
| Scalar coupling constants                                    | Data Type                     | 22.1                               |
| Proton chemical shifts                                       | Data Type                     | 22.1                               |
| Replica exchange calculations                                | Method                        | 22.1                               |
| Thermodynamic integration (TI, in sander)                    | Method                        | 22.1                               |
| Umbrella sampling (in sander)                                | Method                        | 22.1                               |
| Empirical valence bond (EVB) scheme                          | Method                        | 22.1                               |
| QMMM Calculations (in sander)                                | Method                        | 22.1                               |
| Path-integral molecular dynamics (PIMD)                      | Method                        | 22.1, 22.6.10.3                    |
| mdin, mdout, mdinfo, prmtop, inpcrd, refc, mtmd, mdcrd, inptraj, mdvel, mdfrc, mden, restrt, inpdip, rstdip, cpin, cprestrt, cpout, cein, cerestrt, ceout, evbin (sander file types) | File Type                     | 22.2                               |
| Namelist Input Syntax (sander)                               | Syntax/Convention             | 22.4                               |
| &cntrl, &pb, &ewald, &qmmm, &amoeba, &debugf (sander namelists) | Namelist                      | 22.5, 22.11                        |
| &wt namelist (sander varying conditions)                     | Namelist                      | 22.5, 22.9                         |
| Frozen/restrained atoms (sander)                             | Simulation Feature            | 22.6.4                             |
| Belly type dynamics                                          | Method                        | 22.6.4                             |
| XMIN method (minimization)                                   | Method                        | 22.6.5 (ntmin)                     |
| LMOD method (minimization)                                   | Method                        | 22.6.5 (ntmin)                     |
| DL-Find module (minimization)                                | Software Module               | 22.6.5 (ntmin)                     |
| Langevin dynamics                                            | Method                        | 22.6.7 (ntt), 22.6.10.1            |
| Andersen thermostat                                          | Algorithm/Method              | 22.6.7 (ntt), 22.6.10.1, 22.6.10.3 |
| Nose-Hoover chain ensemble (OIN)                             | Method/Ensemble               | 22.6.7 (ntt)                       |
| Stochastic Isokinetic Nose-Hoover RESPA integrator (SINR)    | Algorithm/Method              | 22.6.7 (ntt)                       |
| Berendsen thermostat                                         | Algorithm/Method              | 22.6.7 (ntt=11), 22.6.8, 23.6.6    |
| Berendsen barostat                                           | Algorithm/Method              | 22.6.8                             |
| Monte Carlo barostat                                         | Algorithm/Method              | 22.6.8                             |
| SHAKE bond length constraints                                | Algorithm/Method              | 22.1, 22.6.7, 22.6.9, 22.6.10.3, 22.7.1 |
| "middle" scheme (thermostat)                                 | Method/Framework              | 22.6.7, 22.6.10, 23.2              |
| Velocity Verlet (VV) algorithm                               | Algorithm                     | 22.6.10.1                          |
| Leapfrog algorithm                                           | Algorithm                     | 22.6.10.1                          |
| RATTLE (constraint algorithm for velocities)                 | Algorithm                     | 22.6.10.1, 23.6.6                  |
| SETTLE algorithm                                             | Algorithm                     | 22.6.10.1, 22.6.10.3               |
| PRIMPIMD (primitive PIMD)                                    | Method                        | 22.6.10.2                          |
| Water cap                                                    | Simulation Feature            | 22.6.11                            |
| EMAP restraints                                              | Restraint Type                | 22.6.13                            |
| &emap namelist                                               | Namelist                      | 22.6.13                            |
| Polarizable Gaussian Multipole (pGM) force field             | Model                         | 22.7.1 (ipgm), 22.8                |
| Isotropic Periodic Sum (IPS) method                          | Method                        | 22.7.3, 22.8.1                     |
| &pol_gauss namelist                                          | Namelist                      | 22.8.2                             |
| &dipoles namelist                                            | Namelist                      | 22.7.6                             |
| multisander, multipmemd                                      | Software Feature              | 22.12                              |
| groupfile (for multisander/multipmemd)                       | File Format                   | 22.12                              |
| APBS (as alternate PB solver in Sander)                      | Software/Method               | 22.13                              |
| iAPBS                                                        | Software/Interface            | 22.13                              |
| PQR file format                                              | File Format                   | 22.13                              |
| sander API (Application Programmer Interface)                | Software Interface            | 22.14                              |
| pysander                                                     | Python API for sander         | 22.14                              |
| sanderles API                                                | Software Interface (LES)      | 22.14.2.4, 22.14.4                 |
| pmemd (Particle Mesh Ewald Molecular Dynamics)               | Software                      | 23.                                |
| FEP (Free Energy Perturbation)                               | Method                        | 23.2                               |
| MBAR (Multistate Bennett Acceptance Ratio)                   | Method                        | 23.2                               |
| SPFP, DPFP, SPXP (pmemd.cuda precision models)               | Precision Model               | 23.6, 23.6.3, 23.6.4               |
| CUDA_VISIBLE_DEVICES                                         | Environment Variable          | 23.6.2, 23.6.5                     |
| NCCL (NVIDIA Collective Communications Library)              | Software/Library              | 23.6.4                             |
| Neutral territory method (midpoint for pmemd)                | Algorithm/Method              | 23.5                               |
| Atom and Residue Selections (Amber Masks, NAB Atom Expressions, GROUP Specification) | Syntax/Convention             | 24.                                |
| ambmask (utility)                                            | Software/Tool                 | 24.1                               |
| Self-Guided Langevin dynamics (SGLD)                         | Method/Algorithm              | 25.1                               |
| SGMD (Self-Guided Molecular Dynamics)                        | Method/Algorithm              | 25.1                               |
| SGLD-GLE (SGLD via generalized Langevin equation)            | Method/Algorithm              | 25.1                               |
| RXSGLD (Replica Exchange SGLD)                               | Method/Algorithm              | 25.1                               |
| sgldinfo.sh, sgldwt.sh (scripts for SGLD)                    | Script                        | 25.1                               |
| Accelerated Molecular Dynamics (aMD)                         | Method/Algorithm              | 25.2                               |
| amd.log file                                                 | File Format                   | 25.2.2                             |
| PyReweighting (python toolkit for aMD/GaMD)                  | Software Toolkit              | 25.2.2, 25.3.9                     |
| Gaussian Accelerated Molecular Dynamics (GaMD)               | Method/Algorithm              | 25.3                               |
| LiGaMD (Ligand GaMD)                                         | Method/Algorithm              | 25.3.2                             |
| Pep-GaMD (Peptide GaMD)                                      | Method/Algorithm              | 25.3.3                             |
| LiGaMD3                                                      | Method/Algorithm              | 25.3.4                             |
| PPI-GaMD (Protein-Protein interaction GaMD)                  | Method/Algorithm              | 25.3.5                             |
| gamd.log file                                                | File Format                   | 25.3.6                             |
| Targeted MD                                                  | Method/Algorithm              | 25.4                               |
| Multiply-Targeted MD (MTMD)                                  | Method/Algorithm              | 25.5                               |
| &tgt namelist (MTMD)                                         | Namelist                      | 25.5.1                             |
| Nudged elastic band (NEB) calculations                       | Method/Algorithm              | 25.6                               |
| Partial NEB (PNEB)                                           | Method/Algorithm              | 25.6.1                             |
| Adaptive String Method (ASM)                                 | Method/Algorithm              | 25.7                               |
| Minimum Free Energy Path (MFEP)                              | Concept                       | 25.7                               |
| Collective Variables (CVs for ASM)                           | Parameter/Concept             | 25.7.1                             |
| &asm namelist                                                | Namelist                      | 25.7.1                             |
| Low-MODe (LMOD) methods                                      | Method/Algorithm              | 25.8                               |
| ARPACK                                                       | Software Package              | 25.8                               |
| ZIG-ZAG algorithm (LMOD)                                     | Algorithm                     | 25.8.2                             |
| Co-vibrational low-mode procedure (LMOD)                     | Method                        | 25.8.3                             |
| XMIN (minimization methods: PRCG, LBFGS, TNCG)               | Algorithm/Method              | 25.8.4                             |
| &lmod namelist                                               | Namelist                      | 25.8.4, 25.8.5                     |
| DL-Find Optimization                                         | Method/Software Feature       | 25.9                               |
| libdlfind                                                    | Software/Library              | 25.9                               |
| &dlfind namelist                                             | Namelist                      | 25.9.1                             |
| Thermodynamic integration (TI)                               | Method                        | 26.1                               |
| multisander (for TI)                                         | Software Feature              | 26.1.1                             |
| Softcore Potentials (TI)                                     | Model/Potential               | 26.1.6                             |
| Smoothstep function (softcore TI)                            | Function                      | 26.1.7.1                           |
| Lambda-scheduling (softcore TI)                              | Method                        | 26.1.7.1                           |
| Linear Interaction Energies (LIE)                            | Method                        | 26.2                               |
| Replica Exchange Molecular Dynamics (REMD)                   | Method/Algorithm              | 26.3                               |
| Metropolis Monte Carlo                                       | Algorithm                     | 26.3.1                             |
| T-REMD (Temperature REMD)                                    | Method                        | 26.3.2, 26.3.4                     |
| H-REMD (Hamiltonian REMD)                                    | Method                        | 26.3.2, 26.3.5                     |
| pH-REMD                                                      | Method                        | 26.3.2, 26.3.7                     |
| E-REMD (Redox Potential REMD)                                | Method                        | 26.3.2, 26.3.8                     |
| RXSGLD (Replica Exchange Self-Guided Langevin Dynamics)      | Method                        | 26.3.2, 26.3.6                     |
| Multi-dimensional Replica Exchange                           | Method                        | 26.3.10                            |
| &multirem namelist                                           | Namelist                      | 26.3.10.1                          |
| fixremdcouts.py                                              | Software/Script               | 26.3.10.4                          |
| REAF (Replica Exchange with Arbitrary Degree of Freedom)     | Method                        | 26.3.9                             |
| REST2 (Replica Exchange Solute Tempering 2)                  | Method                        | 26.3.9                             |
| Adaptively Biased Molecular Dynamics (ABMD)                  | Method                        | 26.4                               |
| Potential of mean force (PMF)                                | Concept                       | 26.4.1                             |
| Well-Tempered (WT) ABMD                                      | Method                        | 26.4.1                             |
| Driven ABMD (D-ABMD)                                         | Method                        | 26.4.1                             |
| Swarms-of-Trajectories String Method (STSM)                  | Method                        | 26.4.6                             |
| &colvar namelist (ABMD, SMD, Umbrella Sampling)              | Namelist                      | 26.4.2, 26.4.3, 26.4.4             |
| nfe-umbrella-slice utility                                   | Software/Utility              | 26.4.1, 26.4.8                     |
| Steered Molecular Dynamics (SMD)                             | Method                        | 26.5                               |
| Jarzynski Relationship/Equation                              | Concept/Equation              | 26.5.1                             |
| &smd namelist                                                | Namelist                      | 26.5.2                             |
| Umbrella sampling                                            | Method                        | 26.4.4                             |
| &pmd namelist                                                | Namelist                      | 26.4.4                             |
| EMIL (Absolute Free Energies using EMIL)                     | Method                        | 26.6                               |
| Einstein Molecule                                            | Concept                       | 26.6                               |
| &emil_cntrl namelist                                         | Namelist                      | 26.6.1                             |
| emilParameters.in file                                       | File Format                   | 26.6.2                             |
| Constant pH calculations                                     | Method                        | 27.                                |
| leaprc.constph                                               | Configuration File            | 27.2                               |
| cpin file                                                    | File Format                   | 27.2                               |
| cpinutil.py                                                  | Software/Script               | 27.2                               |
| finddgref.py                                                 | Software/Script               | 27.5.1                             |
| Adaptive solvent QM/MM simulations                           | Method                        | 11.8                               |
| &vsolv namelist (adQM/MM)                                    | Namelist                      | 11.8.2.2                           |
| &adqmmm namelist (adQM/MM)                                   | Namelist                      | 11.8.2.2                           |
| Adaptive buffered force-mixing QM/MM (abfQM/MM)              | Method                        | 11.9                               |
| extended system (abfQM/MM)                                   | System Definition             | 11.9.4.1                           |
| reduced system (abfQM/MM)                                    | System Definition             | 11.9.4.1                           |
| core atoms (abfQM/MM)                                        | Atom Type (abfQM/MM)          | 11.9.4.2                           |
| qm atoms (abfQM/MM)                                          | Atom Type (abfQM/MM)          | 11.9.4.2                           |
| buffer atoms (abfQM/MM)                                      | Atom Type (abfQM/MM)          | 11.9.4.2                           |
| mm atoms (abfQM/MM)                                          | Atom Type (abfQM/MM)          | 11.9.4.2                           |
| &qmmm namelist (for abfQM/MM)                                | Namelist                      | 11.9.5                             |
| ambmask format                                               | Data Format/Syntax            | 11.9.5.1                           |
| Adaptive thermostats (Nose-Hoover, Langevin for abfQM/MM)    | Algorithm/Method              | 11.9.5.2                           |
| SEBOMD (SemiEmpirical Born-Oppenheimer Molecular Dynamics)   | Method/Software Feature       | 11.10                              |
| DivCon                                                       | Software                      | 11.10                              |
| &sebomd namelist                                             | Namelist                      | 11.10.2, 11.10.3                   |
| PIF2, PIF3 (PM3 corrections for SEBOMD)                      | Correction/Model              | 11.10.3 (modif)                    |
| MAIS1, MAIS2 (PM3 corrections for SEBOMD)                    | Correction/Model              | 11.10.3 (modif)                    |
| ReaxFF/AMBER                                                 | Method/Software Feature       | 11.11                              |
| PuReMD (Reactive Molecular Dynamics)                         | Software                      | 11.11                              |
| ReaxFF force fields                                          | Model                         | 11.11.2                            |
| ffield.reaxff                                                | File Format/Default Name      | 11.11.4                            |
| &reaxff namelist                                             | Namelist                      | 11.11.4, 11.11.5                   |
| QEq (Charge equilibration for ReaxFF)                        | Method                        | 11.11.5 (charge_method)            |
| EE (Electronegativity equalization for ReaxFF)               | Method                        | 11.11.5 (charge_method)            |
| ACKS2 (Atom-condensed Kohn-Sham approximated to second order for ReaxFF) | Method                        | 11.11.5 (charge_method)            |
| reaxff-puremd control file                                   | File Format                   | 11.11.6                            |
| External library interface (Amber)                           | Software Feature              | 12.                                |
| MBX library                                                  | Software/Library              | 12.                                |
| KMMD library (Machine Learning Molecular Dynamics)           | Software/Library              | 12.                                |
| &extpot namelist                                             | Namelist                      | 12.2                               |
| PDB Files (Preparation)                                      | Process                       | 13.                                |
| pdb4amber                                                    | Software/Tool                 | 13.                                |
| reduce (program for adding hydrogens)                        | Software/Tool                 | 13., 13.5                          |
| TER record (PDB)                                             | Data Format Element           | 13.1, 13.2, 13.3                   |
| HIE, HID, HIP (Histidine protonation states)                 | Chemical State/Convention     | 13.2                               |
| CYX (Cysteine in disulfide bridge)                           | Chemical State/Convention     | 13.2                               |
| ASH, GLH, LYN (Neutral Asp, Glu, Lys)                        | Chemical State/Convention     | 13.2                               |
| ACE, NHE, NME (Terminal cap residues)                        | Chemical Entity/Convention    | 13.2, 13.3                         |
| AMBER masks                                                  | Syntax/Convention             | 13.3                               |
| NAB atom expressions                                         | Syntax/Convention             | 13.3                               |
| pytleap                                                      | Software/Script               | 13.3                               |
| pdb2pqr                                                      | Software/Tool                 | 13.6                               |
| propka3                                                      | Software (used by pdb2pqr)    | 13.6                               |
| gwh (guess water hydrogens)                                  | Software/Tool                 | 13.7                               |
| packmol-memgen                                               | Software/Tool                 | 13.9                               |
| Memembed                                                     | Software (used by packmol-memgen) | 13.9                               |
| pdbremix                                                     | Software (used by packmol-memgen) | 13.9                               |
| Packmol                                                      | Software (used by packmol-memgen) | 13.9                               |
| charmmlipid2amber.py                                         | Software/Script               | 13.9                               |
| OPM/PPM3 orientation program                                 | Software                      | 13.9                               |
| immers                                                       | Software                      | 13.9                               |
| Lipid ext (Lipid extension force field)                      | Model                         | 13.9.1                             |
| lysophospholipids                                            | Chemical Entity Class         | 13.9.1                             |
| phosphatidylinositol                                         | Chemical Entity Class         | 13.9.1                             |
| cardiolipins                                                 | Chemical Entity Class         | 13.9.1                             |
| sterols (ergosterol, campesterol, sitosterol, stigmasterol)  | Chemical Entity Class         | 13.9.1                             |
| memgen.parm file                                             | File Format                   | 13.9.2                             |
| SIRAH force field (for coarse-graining in packmol-memgen)    | Model                         | 13.9.2 (using --sirah flag)       |
| AMBAT (Amber Membrane Builder and Analysis Tool)             | Software/Tool                 | 13.10                              |
| LEaP (tleap, xleap)                                          | Software/Tool                 | 14.                                |
| Object File Format (OFF)                                     | File Format                   | 14.1                               |
| PREP input files                                             | File Format                   | 14.1                               |
| PARM format parameter sets                                   | File Format                   | 14.1                               |
| loadPdb                                                      | LEaP Command                  | 14.1, 14.4.1, 14.6.29              |
| UNIT (LEaP object)                                           | Data Structure                | 14.2.1, 14.2.3                     |
| RESIDUE (LEaP object)                                        | Data Structure                | 14.2.1, 14.2.3                     |
| ATOM (LEaP object)                                           | Data Structure                | 14.2.1, 14.2.3                     |
| PARMSET (LEaP object)                                        | Data Structure                | 14.2.1, 14.2.3                     |
| leaprc file                                                  | Configuration File            | 14.3, 14.6.7, 14.6.8               |
| Universe Editor (xleap)                                      | Software Component            | 14.3.1                             |
| Unit Editor (xleap)                                          | Software Component            | 14.3.2                             |
| Atom Properties Editor (xleap)                               | Software Component            | 14.3.3                             |
| Parmset Editor (xleap)                                       | Software Component            | 14.3.4                             |
| loadOff                                                      | LEaP Command                  | 14.4.1, 14.6.27                    |
| loadAmberPrep                                                | LEaP Command                  | 14.4.1, 14.6.26                    |
| zMatrix command (LEaP)                                       | LEaP Command                  | 14.4.1, 14.6.49                    |
| loadMol2, loadMol3                                           | LEaP Command                  | 14.4.1, 14.6.28                    |
| addPdbAtomMap                                                | LEaP Command                  | 14.4.1, 14.6.8                     |
| addPdbResMap                                                 | LEaP Command                  | 14.4.1, 14.6.9                     |
| saveAmberParm                                                | LEaP Command                  | 14.4.1, 14.6.35                    |
| saveAmberParmPol                                             | LEaP Command                  | 14.4.1                             |
| NALA, CALA (N-terminal, C-terminal Alanine)                  | Residue Convention            | 14.4.2                             |
| DAN (neutral deoxyadenosine nucleoside)                      | Residue Convention            | 14.4.3                             |
| add (LEaP command)                                           | LEaP Command                  | 14.6.1                             |
| addAtomTypes                                                 | LEaP Command                  | 14.2.3, 14.6.2                     |
| addC4Pairwise                                                | LEaP Command                  | 14.6.3                             |
| addC4Type                                                    | LEaP Command                  | 14.6.4                             |
| addIons, addIons2, addIonsRand                               | LEaP Command                  | 14.6.5, 14.6.6                     |
| addPath                                                      | LEaP Command                  | 14.6.7                             |
| alias (LEaP command)                                         | LEaP Command                  | 14.6.10                            |
| bond (LEaP command)                                          | LEaP Command                  | 14.6.11                            |
| bondByDistance                                               | LEaP Command                  | 14.6.12                            |
| check (LEaP command)                                         | LEaP Command                  | 14.6.13                            |
| combine (LEaP command)                                       | LEaP Command                  | 14.6.14                            |
| copy (LEaP command)                                          | LEaP Command                  | 14.6.15                            |
| createAtom                                                   | LEaP Command                  | 14.6.16                            |
| createResidue                                                | LEaP Command                  | 14.6.17                            |
| createUnit                                                   | LEaP Command                  | 14.6.18                            |
| deleteBond                                                   | LEaP Command                  | 14.6.19                            |
| desc (LEaP command)                                          | LEaP Command                  | 14.2.2, 14.6.20                    |
| groupSelectedAtoms                                           | LEaP Command                  | 14.6.21                            |
| help (LEaP command)                                          | LEaP Command                  | 14.6.22                            |
| impose (LEaP command)                                        | LEaP Command                  | 14.6.23                            |
| list (LEaP command)                                          | LEaP Command                  | 14.2.2, 14.6.24                    |
| loadAmberParams                                              | LEaP Command                  | 14.6.25                            |
| logFile (LEaP command)                                       | LEaP Command                  | 14.3, 14.6.31                      |
| measureGeom                                                  | LEaP Command                  | 14.6.32                            |
| quit (LEaP command)                                          | LEaP Command                  | 14.6.33                            |
| remove (LEaP command)                                        | LEaP Command                  | 14.6.34                            |
| saveMol2                                                     | LEaP Command                  | 14.6.36                            |
| saveOff                                                      | LEaP Command                  | 14.6.37                            |
| savePdb                                                      | LEaP Command                  | 14.6.38                            |
| sequence (LEaP command)                                      | LEaP Command                  | 14.6.39                            |
| set (LEaP command)                                           | LEaP Command                  | 14.2.3, 14.6.40                    |
| OldPrmtopFormat (LEaP set option)                            | LEaP Option                   | 14.6.40                            |
| Dielectric (LEaP set option)                                 | LEaP Option                   | 14.6.40                            |
| PdbWriteCharges (LEaP set option)                            | LEaP Option                   | 14.6.40                            |
| PBRadii (LEaP set option)                                    | LEaP Option                   | 14.6.40                            |
| nocenter (LEaP set option)                                   | LEaP Option                   | 14.6.40                            |
| reorder residues (LEaP set option)                           | LEaP Option                   | 14.6.40                            |
| setBox (LEaP command)                                        | LEaP Command                  | 14.6.41                            |
| solvateBox, solvateOct                                       | LEaP Command                  | 14.6.42                            |
| solvateCap                                                   | LEaP Command                  | 14.6.43                            |
| solvateShell                                                 | LEaP Command                  | 14.6.44                            |
| source (LEaP command)                                        | LEaP Command                  | 14.3, 14.6.45                      |
| transform (LEaP command)                                     | LEaP Command                  | 14.6.46                            |
| translate (LEaP command)                                     | LEaP Command                  | 14.6.47                            |
| verbosity (LEaP command)                                     | LEaP Command                  | 14.3, 14.6.48                      |
| GLYCAM-Web                                                   | Web Tool                      | 14.7                               |
| DMPC (1,2-dimyristoyl-sn-glycero-3-phosphocholine)           | Chemical Entity               | 14.7.2                             |
| Solvate.sh                                                   | Script                        | 14.7.4                             |
| Amber parameter files (*.dat, frcmod.*)                      | File Format                   | 15.                                |
| ParmEd                                                       | Software/Tool                 | 15., 15.2                          |
| parameter-topology file (parmtop)                            | File Format                   | 15.1.2                             |
| coordinate file (*.crd, PDB)                                 | File Format                   | 15.1.2 (footnote)                  |
| Lennard-Jones potential (6-12)                               | Model/Equation                | 15.1.7                             |
| Lorentz/Berthelot mixing rules                               | Rule/Convention               | 15.1.7                             |
| AM1-BBC method (charge calculation)                          | Method                        | 15.1.7                             |
| addAtomicNumber (ParmEd command)                             | ParmEd Command                | 15.2.2.1                           |
| addDihedral (ParmEd command)                                 | ParmEd Command                | 15.2.2.2                           |
| addExclusions (ParmEd command)                               | ParmEd Command                | 15.2.2.3                           |
| addLJType (ParmEd command)                                   | ParmEd Command                | 15.2.2.4                           |
| addPDB (ParmEd command)                                      | ParmEd Command                | 15.2.2.5                           |
| add12_6_4 (ParmEd command)                                   | ParmEd Command                | 15.2.2.6                           |
| chamber (ParmEd command)                                     | ParmEd Command                | 15.2.2.8                           |
| CHARMM force field (via chamber)                             | Model                         | 15.2.2.8                           |
| Urey-Bradley term                                            | Model Term                    | 15.2.2.8                           |
| CMAP (Correction Map)                                        | Model Term                    | 15.2.2.8                           |
| CTITLE, FORCE_FIELD_TYPE, CHARMM_UREY_BRADLEY_COUNT etc. (ParmEd %FLAGs for CHARMM) | File Format Element           | 15.2.2.8                           |
| change (ParmEd command)                                      | ParmEd Command                | 15.2.2.9                           |
| changeLJPair (ParmEd command)                                | ParmEd Command                | 15.2.2.10                          |
| changeLJ14Pair (ParmEd command)                              | ParmEd Command                | 15.2.2.11                          |
| changeLJSingleType (ParmEd command)                          | ParmEd Command                | 15.2.2.12                          |
| changeProtState (ParmEd command)                             | ParmEd Command                | 15.2.2.13                          |
| changeRedoxState (ParmEd command)                            | ParmEd Command                | 15.2.2.14                          |
| changeRadii (ParmEd command)                                 | ParmEd Command                | 15.2.2.15                          |
| checkValidity (ParmEd command)                               | ParmEd Command                | 15.2.2.16                          |
| defineSolvent (ParmEd command)                               | ParmEd Command                | 15.2.2.17                          |
| deleteBond (ParmEd command)                                  | ParmEd Command                | 15.2.2.18                          |
| deleteDihedral (ParmEd command)                              | ParmEd Command                | 15.2.2.19                          |
| deletePDB (ParmEd command)                                   | ParmEd Command                | 15.2.2.20                          |
| energy (ParmEd command)                                      | ParmEd Command                | 15.2.2.21                          |
| OpenMM                                                       | Software/API                  | 15.2.2.21, 15.2.2.33, 15.2.2.35    |
| go (ParmEd command)                                          | ParmEd Command                | 15.2.2.22                          |
| gromber (ParmEd command)                                     | ParmEd Command                | 15.2.2.23                          |
| HMassRepartition (ParmEd command)                            | ParmEd Command                | 15.2.2.24                          |
| history (ParmEd command)                                     | ParmEd Command                | 15.2.2.26                          |
| interpolate (ParmEd command)                                 | ParmEd Command                | 15.2.2.27                          |
| listParms (ParmEd command)                                   | ParmEd Command                | 15.2.2.28                          |
| lmod (ParmEd command)                                        | ParmEd Command                | 15.2.2.29                          |
| loadCoordinates (ParmEd command)                             | ParmEd Command                | 15.2.2.30                          |
| loadRestrt (ParmEd command)                                  | ParmEd Command                | 15.2.2.21, 15.2.2.31, 15.2.2.38, 15.2.2.40 |
| ls (ParmEd command)                                          | ParmEd Command                | 15.2.2.32                          |
| minimize (ParmEd command)                                    | ParmEd Command                | 15.2.2.33                          |
| netCharge (ParmEd command)                                   | ParmEd Command                | 15.2.2.34                          |
| outCIF (ParmEd command)                                      | ParmEd Command                | 15.2.2.36                          |
| PDBx/mmCIF file format                                       | File Format                   | 15.2.2.30, 15.2.2.36               |
| outparm (ParmEd command)                                     | ParmEd Command                | 15.2.2.37                          |
| outPDB (ParmEd command)                                      | ParmEd Command                | 15.2.2.38                          |
| parm (ParmEd command)                                        | ParmEd Command                | 15.2.2.39                          |
| parmout (ParmEd command)                                     | ParmEd Command                | 15.2.2.40                          |
| printAngles (ParmEd command)                                 | ParmEd Command                | 15.2.2.41                          |
| printBonds (ParmEd command)                                  | ParmEd Command                | 15.2.2.42                          |
| printDetails (ParmEd command)                                | ParmEd Command                | 15.2.2.43                          |
| printDihedrals (ParmEd command)                              | ParmEd Command                | 15.2.2.44                          |
| printFlags (ParmEd command)                                  | ParmEd Command                | 15.2.2.45                          |
| printInfo (ParmEd command)                                   | ParmEd Command                | 15.2.2.46                          |
| printLJMatrix (ParmEd command)                               | ParmEd Command                | 15.2.2.47                          |
| printLJTypes (ParmEd command)                                | ParmEd Command                | 15.2.2.48                          |
| printPointers (ParmEd command)                               | ParmEd Command                | 15.2.2.49                          |
| quit (ParmEd command)                                        | ParmEd Command                | 15.2.2.50                          |
| scale (ParmEd command)                                       | ParmEd Command                | 15.2.2.51                          |
| scee (ParmEd command)                                        | ParmEd Command                | 15.2.2.52                          |
| scnb (ParmEd command)                                        | ParmEd Command                | 15.2.2.53                          |
| setAngle (ParmEd command)                                    | ParmEd Command                | 15.2.2.54                          |
| setBond (ParmEd command)                                     | ParmEd Command                | 15.2.2.18, 15.2.2.55               |
| setMolecules (ParmEd command)                                | ParmEd Command                | 15.2.2.56                          |
| setOverwrite (ParmEd command)                                | ParmEd Command                | 15.2.2.57                          |
| source (ParmEd command)                                      | ParmEd Command                | 15.2.2.58                          |
| strip (ParmEd command)                                       | ParmEd Command                | 15.2.2.37, 15.2.2.59               |
| summary (ParmEd command)                                     | ParmEd Command                | 15.2.2.60                          |
| tiMerge (ParmEd command)                                     | ParmEd Command                | 15.2.2.61                          |
| writeFrcmod (ParmEd command)                                 | ParmEd Command                | 15.2.2.62                          |
| writeOFF (ParmEd command)                                    | ParmEd Command                | 15.2.2.63                          |
| Interactive Python Shell (ParmEd)                            | Software Feature              | 15.2.5.1                           |
| Antechamber                                                  | Software Suite                | 16.                                |
| GAFF (General Amber Force Field)                             | Model                         | 16.                                |
| gaff.dat, gaff2.dat                                          | File/Parameter Set            | 16.                                |
| pyMSMT (Python Metal Site Modeling Toolbox)                  | Software Suite                | 16., 19.                           |
| HF/6-31G* RESP (charge method)                               | Method                        | 16.                                |
| AM1-BCC (charge method)                                      | Method                        | 16., 16.1.2, 16.3.1, 16.3.2        |
| ABCG2 (charge method)                                        | Method                        | 16.                                |
| PARMSCAN                                                     | Software Package              | 16.                                |
| antechamber (program)                                        | Software                      | 16.1.1                             |
| sqm (program, for antechamber)                               | Software                      | 16.1.1, 16.3.1                     |
| mopac (program, for antechamber)                             | Software                      | 16.1.1                             |
| divcon (program, for antechamber)                            | Software                      | 16.1.1                             |
| atomtype (program)                                           | Software                      | 16.1.1, 16.3.1                     |
| bondtype (program)                                           | Software                      | 16.1.1, 16.3.3, 16.4.1             |
| espgen (program)                                             | Software                      | 16.1.1, 16.3.5, 19.2.6, 20.2       |
| respgen (program)                                            | Software                      | 16.1.1, 16.3.6                     |
| prepgen (program)                                            | Software                      | 16.1.1, 16.3.4, 16.4.3             |
| parmchk2 (program)                                           | Software                      | 16.1.1, 16.1.2                     |
| PARMCHK.DAT                                                  | File/Parameter Set            | 16.1.2                             |
| ATCOR.DAT                                                    | File/Parameter Set            | 16.1.2                             |
| acdoctor (program)                                           | Software                      | 16.3.1, 16.4.1                     |
| parmcal (program)                                            | Software                      | 16.4.2                             |
| residuegen (program)                                         | Software                      | 16.4.3                             |
| match (program)                                              | Software                      | 16.4.4                             |
| match_atomname (program)                                     | Software                      | 16.4.5                             |
| paramfit                                                     | Software                      | 17.                                |
| Job Control File (paramfit)                                  | File Format                   | 17.2                               |
| Genetic Algorithm (paramfit)                                 | Algorithm                     | 17.2.4, 17.2.5                     |
| Simplex Algorithm (paramfit)                                 | Algorithm                     | 17.2.4, 17.2.5                     |
| mdgx                                                         | Software                      | 18.                                |
| IPolQ (Implicitly Polarized Charge model)                    | Model                         | 18., 18.1, 18.4                    |
| RESP (Restrained ElectroStatic Potential)                    | Method/Algorithm              | 18.3, 20.                          |
| Virtual Site Support (mdgx)                                  | Software Feature              | 18.5                               |
| CMAP surfaces (fitting in mdgx)                              | Model Term/Parameter          | 18.6                               |
| Configuration Sampling (mdgx)                                | Method                        | 18.7                               |
| Peptide multi-simulator (mdgx GPU)                           | Software Feature              | 18.8                               |
| RATTLE (constraint algorithm)                                | Algorithm                     | 18.8                               |
| I/r-RESPA (multi-time stepping)                              | Algorithm                     | 18.8                               |
| MCPB.py (Metal Center Parameter Builder)                     | Software                      | 19.1, 19.2.1                       |
| IPMach.py (Ion Parameterization Machine)                     | Software                      | 19.1, 19.2.2                       |
| PdbSearcher.py                                               | Software                      | 19.1, 19.2.3                       |
| OptC4.py                                                     | Software                      | 19.1, 19.2.4                       |
| CartHess2FC.py                                               | Software                      | 19.1, 19.2.5                       |
| ProScrs.py (Protein Scissors)                                | Software                      | 19.1, 19.2.7                       |
| car_to_files.py                                              | Software                      | 19.1, 19.2.8                       |
| mol2rtf.py                                                   | Software                      | 19.1, 19.2.9                       |
| metalpdb2mol2.py                                             | Software                      | 19.1, 19.2.10                      |
| py_resp.py                                                   | Software                      | 20.                                |
| RESP-ind model                                               | Model                         | 20., 20.2, 20.3.1, 20.4.5          |
| RESP-perm model                                              | Model                         | 20., 20.2, 20.3.1, 20.4.5          |
| RESP-perm-v model                                            | Model                         | 20., 20.3.1                        |
| pyresp_gen.py                                                | Software                      | 20.1                               |
| pGM damping function                                         | Function/Model Feature        | 20.1.1, 20.2                       |
| PyPE_RESP                                                    | Software/Pipeline             | 20.4                               |
| RDKit                                                        | Software Package              | 20.4.2                             |
| SMARTS patterns                                              | Syntax/Convention             | 20.4.2                             |
| UnitCell (program for crystal simulations)                   | Software                      | 21.1                               |
| PropPDB (program for crystal simulations)                    | Software                      | 21.2                               |
| AddToBox (program for crystal simulations)                   | Software                      | 21.3                               |
| ChBox (program for crystal simulations)                      | Software                      | 21.4                               |
| sander (simulation engine)                                   | Software                      | 22.                                |
| Particle-mesh Ewald (PME)                                    | Algorithm/Method              | 22.1, 22.7.2                       |
| NVE ensemble                                                 | Simulation Condition          | 22.1, 22.6.7, 22.7.5, 22.8.1       |
| NVT ensemble                                                 | Simulation Condition          | 22.6.2 (nmropt, implicit), 22.8.1  |
| NPT ensemble                                                 | Simulation Condition          | 22.6.8, 22.8.1, 23.2, 23.6.6     |
| Generalized Born/surface area model (in sander)              | Model                         | 22.1                               |
| NOESY intensities                                            | Data Type                     | 22.1                               |
| Residual dipolar couplings                                   | Data Type                     | 22.1                               |
| Scalar coupling constants                                    | Data Type                     | 22.1                               |
| Proton chemical shifts                                       | Data Type                     | 22.1                               |
| Replica exchange calculations                                | Method                        | 22.1                               |
| Thermodynamic integration (TI, in sander)                    | Method                        | 22.1                               |
| Umbrella sampling (in sander)                                | Method                        | 22.1                               |
| Empirical valence bond (EVB) scheme                          | Method                        | 22.1                               |
| QMMM Calculations (in sander)                                | Method                        | 22.1                               |
| Path-integral molecular dynamics (PIMD)                      | Method                        | 22.1, 22.6.10.3                    |
| mdin, mdout, mdinfo, prmtop, inpcrd, refc, mtmd, mdcrd, inptraj, mdvel, mdfrc, mden, restrt, inpdip, rstdip, cpin, cprestrt, cpout, cein, cerestrt, ceout, evbin (sander file types) | File Type                     | 22.2                               |
| Namelist Input Syntax (sander)                               | Syntax/Convention             | 22.4                               |
| &cntrl, &pb, &ewald, &qmmm, &amoeba, &debugf (sander namelists) | Namelist                      | 22.5, 22.11                        |
| &wt namelist (sander varying conditions)                     | Namelist                      | 22.5, 22.9                         |
| Frozen/restrained atoms (sander)                             | Simulation Feature            | 22.6.4                             |
| Belly type dynamics                                          | Method                        | 22.6.4                             |
| XMIN method (minimization)                                   | Method                        | 22.6.5 (ntmin)                     |
| LMOD method (minimization)                                   | Method                        | 22.6.5 (ntmin)                     |
| DL-Find module (minimization)                                | Software Module               | 22.6.5 (ntmin)                     |
| Langevin dynamics                                            | Method                        | 22.6.7 (ntt), 22.6.10.1            |
| Andersen thermostat                                          | Algorithm/Method              | 22.6.7 (ntt), 22.6.10.1, 22.6.10.3 |
| Nose-Hoover chain ensemble (OIN)                             | Method/Ensemble               | 22.6.7 (ntt)                       |
| Stochastic Isokinetic Nose-Hoover RESPA integrator (SINR)    | Algorithm/Method              | 22.6.7 (ntt)                       |
| Berendsen thermostat                                         | Algorithm/Method              | 22.6.7 (ntt=11), 22.6.8, 23.6.6    |
| Berendsen barostat                                           | Algorithm/Method              | 22.6.8                             |
| Monte Carlo barostat                                         | Algorithm/Method              | 22.6.8                             |
| SHAKE bond length constraints                                | Algorithm/Method              | 22.1, 22.6.7, 22.6.9, 22.6.10.3, 22.7.1 |
| "middle" scheme (thermostat)                                 | Method/Framework              | 22.6.7, 22.6.10, 23.2              |
| Velocity Verlet (VV) algorithm                               | Algorithm                     | 22.6.10.1                          |
| Leapfrog algorithm                                           | Algorithm                     | 22.6.10.1                          |
| RATTLE (constraint algorithm for velocities)                 | Algorithm                     | 22.6.10.1, 23.6.6                  |
| SETTLE algorithm                                             | Algorithm                     | 22.6.10.1, 22.6.10.3               |
| PRIMPIMD (primitive PIMD)                                    | Method                        | 22.6.10.2                          |
| Water cap                                                    | Simulation Feature            | 22.6.11                            |
| EMAP restraints                                              | Restraint Type                | 22.6.13                            |
| &emap namelist                                               | Namelist                      | 22.6.13                            |
| Polarizable Gaussian Multipole (pGM) force field             | Model                         | 22.7.1 (ipgm), 22.8                |
| Isotropic Periodic Sum (IPS) method                          | Method                        | 22.7.3, 22.8.1                     |
| &pol_gauss namelist                                          | Namelist                      | 22.8.2                             |
| &dipoles namelist                                            | Namelist                      | 22.7.6                             |
| multisander, multipmemd                                      | Software Feature              | 22.12                              |
| groupfile (for multisander/multipmemd)                       | File Format                   | 22.12                              |
| APBS (as alternate PB solver in Sander)                      | Software/Method               | 22.13                              |
| iAPBS                                                        | Software/Interface            | 22.13                              |
| PQR file format                                              | File Format                   | 22.13                              |
| sander API (Application Programmer Interface)                | Software Interface            | 22.14                              |
| pysander                                                     | Python API for sander         | 22.14                              |
| sanderles API                                                | Software Interface (LES)      | 22.14.2.4, 22.14.4                 |
| pmemd (Particle Mesh Ewald Molecular Dynamics)               | Software                      | 23.                                |
| FEP (Free Energy Perturbation)                               | Method                        | 23.2                               |
| MBAR (Multistate Bennett Acceptance Ratio)                   | Method                        | 23.2                               |
| SPFP, DPFP, SPXP (pmemd.cuda precision models)               | Precision Model               | 23.6, 23.6.3, 23.6.4               |
| CUDA_VISIBLE_DEVICES                                         | Environment Variable          | 23.6.2, 23.6.5                     |
| NCCL (NVIDIA Collective Communications Library)              | Software/Library              | 23.6.4                             |
| Neutral territory method (midpoint for pmemd)                | Algorithm/Method              | 23.5                               |
| Atom and Residue Selections (Amber Masks, NAB Atom Expressions, GROUP Specification) | Syntax/Convention             | 24.                                |
| ambmask (utility)                                            | Software/Tool                 | 24.1                               |
| Self-Guided Langevin dynamics (SGLD)                         | Method/Algorithm              | 25.1                               |
| SGMD (Self-Guided Molecular Dynamics)                        | Method/Algorithm              | 25.1                               |
| SGLD-GLE (SGLD via generalized Langevin equation)            | Method/Algorithm              | 25.1                               |
| RXSGLD (Replica Exchange SGLD)                               | Method/Algorithm              | 25.1, 26.3.6                       |
| sgldinfo.sh, sgldwt.sh (scripts for SGLD)                    | Script                        | 25.1                               |
| Accelerated Molecular Dynamics (aMD)                         | Method/Algorithm              | 25.2                               |
| amd.log file                                                 | File Format                   | 25.2.2                             |
| PyReweighting (python toolkit for aMD/GaMD)                  | Software Toolkit              | 25.2.2, 25.3.9                     |
| Gaussian Accelerated Molecular Dynamics (GaMD)               | Method/Algorithm              | 25.3                               |
| LiGaMD (Ligand GaMD)                                         | Method/Algorithm              | 25.3.2                             |
| Pep-GaMD (Peptide GaMD)                                      | Method/Algorithm              | 25.3.3                             |
| LiGaMD3                                                      | Method/Algorithm              | 25.3.4                             |
| PPI-GaMD (Protein-Protein interaction GaMD)                  | Method/Algorithm              | 25.3.5                             |
| gamd.log file                                                | File Format                   | 25.3.6                             |
| Targeted MD                                                  | Method/Algorithm              | 25.4                               |
| Multiply-Targeted MD (MTMD)                                  | Method/Algorithm              | 25.5                               |
| &tgt namelist (MTMD)                                         | Namelist                      | 25.5.1                             |
| Nudged elastic band (NEB) calculations                       | Method/Algorithm              | 25.6                               |
| Partial NEB (PNEB)                                           | Method/Algorithm              | 25.6.1                             |
| Adaptive String Method (ASM)                                 | Method/Algorithm              | 25.7                               |
| Minimum Free Energy Path (MFEP)                              | Concept                       | 25.7                               |
| Collective Variables (CVs for ASM)                           | Parameter/Concept             | 25.7.1                             |
| &asm namelist                                                | Namelist                      | 25.7.1                             |
| Low-MODe (LMOD) methods                                      | Method/Algorithm              | 25.8                               |
| ARPACK                                                       | Software Package              | 25.8                               |
| ZIG-ZAG algorithm (LMOD)                                     | Algorithm                     | 25.8.2                             |
| Co-vibrational low-mode procedure (LMOD)                     | Method                        | 25.8.3                             |
| XMIN (minimization methods: PRCG, LBFGS, TNCG)               | Algorithm/Method              | 25.8.4                             |
| &lmod namelist                                               | Namelist                      | 25.8.4, 25.8.5                     |
| DL-Find Optimization                                         | Method/Software Feature       | 25.9                               |
| libdlfind                                                    | Software/Library              | 25.9                               |
| &dlfind namelist                                             | Namelist                      | 25.9.1                             |
| Thermodynamic integration (TI)                               | Method                        | 26.1                               |
| multisander (for TI)                                         | Software Feature              | 26.1.1                             |
| Softcore Potentials (TI)                                     | Model/Potential               | 26.1.6                             |
| Smoothstep function (softcore TI)                            | Function                      | 26.1.7.1                           |
| Lambda-scheduling (softcore TI)                              | Method                        | 26.1.7.1                           |
| Linear Interaction Energies (LIE)                            | Method                        | 26.2                               |
| Replica Exchange Molecular Dynamics (REMD)                   | Method/Algorithm              | 26.3                               |
| Metropolis Monte Carlo                                       | Algorithm                     | 26.3.1                             |
| T-REMD (Temperature REMD)                                    | Method                        | 26.3.2, 26.3.4                     |
| H-REMD (Hamiltonian REMD)                                    | Method                        | 26.3.2, 26.3.5                     |
| pH-REMD                                                      | Method                        | 26.3.2, 26.3.7                     |
| E-REMD (Redox Potential REMD)                                | Method                        | 26.3.2, 26.3.8                     |
| RXSGLD (Replica Exchange Self-Guided Langevin Dynamics)      | Method                        | 26.3.2, 26.3.6                     |
| Multi-dimensional Replica Exchange                           | Method                        | 26.3.10                            |
| &multirem namelist                                           | Namelist                      | 26.3.10.1                          |
| fixremdcouts.py                                              | Software/Script               | 26.3.10.4                          |
| REAF (Replica Exchange with Arbitrary Degree of Freedom)     | Method                        | 26.3.9                             |
| REST2 (Replica Exchange Solute Tempering 2)                  | Method                        | 26.3.9                             |
| Adaptively Biased Molecular Dynamics (ABMD)                  | Method                        | 26.4                               |
| Potential of mean force (PMF)                                | Concept                       | 26.4.1                             |
| Well-Tempered (WT) ABMD                                      | Method                        | 26.4.1                             |
| Driven ABMD (D-ABMD)                                         | Method                        | 26.4.1                             |
| Swarms-of-Trajectories String Method (STSM)                  | Method                        | 26.4.6                             |
| &colvar namelist (ABMD, SMD, Umbrella Sampling)              | Namelist                      | 26.4.2, 26.4.3, 26.4.4             |
| nfe-umbrella-slice utility                                   | Software/Utility              | 26.4.1, 26.4.8                     |
| Steered Molecular Dynamics (SMD)                             | Method                        | 26.5                               |
| Jarzynski Relationship/Equation                              | Concept/Equation              | 26.5.1                             |
| &smd namelist                                                | Namelist                      | 26.5.2                             |
| Umbrella sampling                                            | Method                        | 26.4.4                             |
| &pmd namelist                                                | Namelist                      | 26.4.4                             |
| EMIL (Absolute Free Energies using EMIL)                     | Method                        | 26.6                               |
| Einstein Molecule                                            | Concept                       | 26.6                               |
| &emil_cntrl namelist                                         | Namelist                      | 26.6.1                             |
| emilParameters.in file                                       | File Format                   | 26.6.2                             |
| Constant pH calculations                                     | Method                        | 27.                                |
| leaprc.constph                                               | Configuration File            | 27.2                               |
| cpin file                                                    | File Format                   | 27.2                               |
| cpinutil.py                                                  | Software/Script               | 27.2, 27.5.1, 28.1                 |
| finddgref.py                                                 | Software/Script               | 27.5.1                             |
| pH Replica Exchange MD (pH-REMD)                             | Method                        | 27.6                               |
| cphstats                                                     | Software/Tool                 | 27.7                               |
| Constant Redox Potential calculations                        | Method                        | 28.                                |
| leaprc.conste                                                | Configuration File            | 28.1                               |
| HEH (redox-active heme residue)                              | Residue Convention            | 28.1, 28.4                         |
| cein file                                                    | File Format                   | 28.1                               |
| ceinutil.py                                                  | Software/Script               | 28.1                               |
| Redox Potential Replica Exchange MD (E-REMD)                 | Method                        | 28.5                               |
| cestats                                                      | Software/Tool                 | 28.6                               |
| Continuous constant pH molecular dynamics (CpHMD)            | Method                        | 29.                                |
| λ-dynamics                                                   | Method/Framework              | 29.                                |
| phmd.lib                                                     | Library File                  | 29.2                               |
| frcmod.phmd                                                  | Frcmod File                   | 29.2                               |
| &phmdin namelist                                             | Namelist                      | 29.2.1                             |
| &phmdparm namelist                                           | Namelist                      | 29.2.2                             |
| &phmdstrt namelist                                           | Namelist                      | 29.2.3                             |
| walker_extraction (script for CpHMD pH-REMD)                 | Script                        | 29.3                               |
| wrap_traj (script for CpHMD pH-REMD)                         | Script                        | 29.3                               |
| renameLamb.sh (script for CpHMD pH-REMD)                     | Script                        | 29.3                               |
| getS_fitpKa_plot_taut.sh (script for CpHMD pH-REMD)          | Script                        | 29.3                               |
| NMR refinement                                               | Process/Method                | 30.                                |
| DISANG file                                                  | File Format                   | 30.1                               |
| &rst namelist (NMR restraints)                               | Namelist                      | 30.1.1                             |
| makeDIST_RST, makeANG_RST, makeCHIR_RST, makeDIP_RST (NMR restraint prep tools) | Software/Tool                 | 30.1, 30.7.1, 30.7.2, 30.7.3, 30.7.4 |
| &noeexp namelist (NOESY restraints)                          | Namelist                      | 30.2                               |
| &shf namelist (chemical shift restraints)                    | Namelist                      | 30.3                               |
| &pcshift namelist (pseudocontact shift restraints)           | Namelist                      | 30.4                               |
| &align namelist (direct dipolar coupling restraints)         | Namelist                      | 30.5                               |
| &csa namelist (residual CSA restraints)                      | Namelist                      | 30.6                               |
| fantasian (program for magnetic anisotropy tensor)           | Software                      | 30.7.6                             |
| sviol, sviol2, senergy (NMR analysis scripts)                | Script                        | 30.1, 30.8                         |
| Xray and cryoEM refinement                                   | Process/Method                | 31.                                |
| EMAP restraints                                              | Restraint Type                | 31.1                               |
| &emap namelist                                               | Namelist                      | 31.1                               |
| FRETrest (Forster Resonance Energy Transfer restraints)      | Method/Script Set             | 31.2                               |
| placeAV.py, FRETrest.py (FRETrest scripts)                   | Script                        | 31.2                               |
| Structure factor calculations (Xray/cryoEM)                  | Method                        | 31.3.1                             |
| &xray namelist                                               | Namelist                      | 31.3.3                             |
| Locally-enhanced sampling (LES)                              | Method                        | 32.                                |
| ADDLES module                                                | Software Module               | 32.1, 32.2, 32.3                   |
| MOIL-View                                                    | Software (for LES analysis)   | 32.1                               |
| gem.pmemd                                                    | Software                      | 33.                                |
| Amoeba force field                                           | Model                         | 33.1                               |
| GEM* water potential                                         | Model                         | 33.1                               |
| &amoeba namelist                                             | Namelist                      | 33.1, 33.2                         |
| &gem namelist                                                | Namelist                      | 33.1, 33.2                         |
| Artificial Intelligence / Machine Learning                   | Method Class                  | 34.                                |
| KMMD (Molecular Dynamics Using a Kernel Machine)             | Method/Software Feature       | 34.1                               |
| Kernel Modified Molecular Dynamics (KMMD)                    | Method                        | 34.1                               |
| NRBF (Normalised Radial Basis Function) Network/Kernel Machine | Model/Algorithm               | 34.1                               |
| &extpot namelist (for KMMD)                                  | Namelist                      | 34.1.2                             |
| .json-like file (KMMD control)                               | File Format                   | 34.1.2                             |
| NAB (Nucleic Acid Builder)                                   | Software/Language             | 35.                                |
| nabc (NAB C interface)                                       | Software/Interface            | 35.                                |
| libsff (simple force field library)                          | Software/Library              | 35.                                |
| mdout_analyzer.py                                            | Software/Tool                 | 36.                                |
| ambpdb                                                       | Software/Tool                 | 36.1                               |
| cpptraj                                                      | Software/Tool                 | 37.                                |
| Atom Mask Selection Syntax (cpptraj)                         | Syntax/Convention             | 37.3.2                             |
| Parameter/Reference Tagging (cpptraj)                        | Feature/Convention            | 37.3.4                             |
| COORDS Data Set Commands (cpptraj)                           | Command Set                   | 37.7                               |
| TRAJ data set (cpptraj)                                      | Data Set Type                 | 37.7                               |
| General Commands (cpptraj)                                   | Command Set                   | 37.8                               |
| Topology File Commands (cpptraj)                             | Command Set                   | 37.9                               |
| Trajectory File Commands (cpptraj)                           | Command Set                   | 37.10                              |
| Action Commands (cpptraj)                                    | Command Set                   | 37.11                              |
| `lie` (cpptraj command)                                              | Method                                    | 37.11.45. lie                             |
| `<Ligand mask>`                                                      | Identifier                                | 37.11.45. lie                             |
| `<Surroundings mask>`                                                | Identifier                                | 37.11.45. lie                             |
| `nopbc`                                                              | Parameter                                 | 37.11.45. lie                             |
| `EELEC` (Electrostatic energy)                                       | Measure/value                             | 37.11.45. lie                             |
| `EVDW` (van der Waals energy)                                        | Measure/value                             | 37.11.45. lie                             |
| `kcal/mol`                                                           | Unit                                      | 37.11.45. lie                             |
| `Angstroms`                                                          | Unit                                      | 37.11.45. lie                             |
| shifting function (for electrostatics)                               | Method                                    | 37.11.45. lie                             |
| minimum image convention                                             | Method / Convention                       | 37.11.45. lie                             |
| `lipidorder` (cpptraj command)                                       | Method                                    | 37.11.46. lipidorder                      |
| order parameter (Sx, Sy, Sz, \|SCD\|)                                | Measure/value                             | 37.11.46. lipidorder                      |
| `|SCD|` (deuterium-order parameter)                                   | Measure/value                             | 37.11.46. lipidorder                      |
| C-H vectors                                                          | Chemical Entity / Identifier              | 37.11.46. lipidorder                      |
| `lipidscd` (cpptraj command)                                         | Method                                    | 37.11.47. lipidscd                        |
| `<P2>` (raw order parameter values)                                  | Measure/value                             | 37.11.47. lipidscd                        |
| carboxyl groups                                                      | Chemical Entity                           | 37.11.47. lipidscd                        |
| `makestructure` (cpptraj command)                                    | Method                                    | 37.11.48. makestructure                   |
| phi/psi backbone angles                                              | Measure/value                             | 37.11.48. makestructure                   |
| alpha (secondary structure)                                          | State                                     | 37.11.48. makestructure                   |
| extended (secondary structure)                                       | State                                     | 37.11.48. makestructure                   |
| typeI turn                                                           | State                                     | 37.11.48. makestructure                   |
| `<dih type>` (dihedral type identifier)                              | Identifier                                | 37.11.48. makestructure                   |
| `mask` (cpptraj command)                                             | Method                                    | 37.11.49. mask                            |
| Atom mask                                                            | Identifier                                | 37.11.49. mask                            |
| `matrix` (cpptraj command)                                           | Method                                    | 37.11.50. matrix                          |
| Distance matrix                                                      | Data Format / Measure/value               | 37.11.50. matrix (dist)                   |
| Correlation matrix (dynamic cross correlation)                       | Data Format / Measure/value               | 37.11.50. matrix (correl)                 |
| Coordinate covariance matrix                                         | Data Format / Measure/value               | 37.11.50. matrix (covar)                  |
| Mass-weighted coordinate covariance matrix                           | Data Format / Measure/value               | 37.11.50. matrix (mwcovar)                |
| Isotropically Distributed Ensemble Analysis (IDEA) matrix            | Method / Data Format                      | 37.11.50. matrix (idea)                   |
| Isotropic Reorientational Eigenmode Dynamics (IRED) matrix           | Method / Data Format                      | 37.11.50. matrix (ired)                   |
| Legendre polynomials                                                 | Mathematical Concept                      | 37.11.50. matrix                          |
| `mindist` (cpptraj command)                                          | Method                                    | 37.11.51. mindist/maxdist                 |
| `maxdist` (cpptraj command)                                          | Method                                    | 37.11.51. mindist/maxdist                 |
| `minimage` (cpptraj command)                                         | Method                                    | 37.11.52. minimage                        |
| `molsurf` (cpptraj command)                                          | Method                                    | 37.11.53. molsurf                         |
| Connolly surface area                                                | Measure/value                             | 37.11.53. molsurf                         |
| `multidihedral` (cpptraj command)                                    | Method                                    | 37.11.54. multidihedral                   |
| `multipucker` (cpptraj command)                                      | Method                                    | 37.11.55. multipucker                     |
| Altona & Sundaralingam method (pucker)                               | Method                                    | 37.11.55. multipucker                     |
| Cremer and Pople method (pucker)                                     | Method                                    | 37.11.55. multipucker                     |
| `multivector` (cpptraj command)                                      | Method                                    | 37.11.56. multivector                     |
| `nastruct` (cpptraj command)                                         | Method                                    | 37.11.57. nastruct                        |
| nucleic acid base reference                                          | Data Source / Convention                  | 37.11.57. nastruct                        |
| 3DNA (NA parameters convention)                                      | Method / Convention                       | 37.11.57. nastruct                        |
| base pair parameters (shear, stretch, etc.)                          | Measure/value                             | 37.11.57. nastruct                        |
| base pair step parameters (shift, slide, etc.)                       | Measure/value                             | 37.11.57. nastruct                        |
| helical parameters (X-displacement, rise, etc.)                      | Measure/value                             | 37.11.57. nastruct                        |
| sugar pucker                                                         | Measure/value                             | 37.11.57. nastruct                        |
| `nativecontacts` (cpptraj command)                                   | Method                                    | 37.11.58. nativecontacts                  |
| native contact                                                       | State / Measure/value                     | 37.11.58. nativecontacts                  |
| non-native contact                                                   | State / Measure/value                     | 37.11.58. nativecontacts                  |
| contact map                                                          | Data Format / Measure/value               | 37.11.58. nativecontacts                  |
| `outtraj` (cpptraj command)                                          | Method                                    | 37.11.59. outtraj                         |
| `pairdist` (cpptraj command)                                         | Method                                    | 37.11.60. pairdist                        |
| pair distribution function                                           | Measure/value                             | 37.11.60. pairdist                        |
| `pairwise` (cpptraj command)                                         | Method                                    | 37.11.61. pairwise                        |
| `principal` (cpptraj command)                                        | Method                                    | 37.11.62. principal                       |
| principal axes                                                       | Measure/value                             | 37.11.62. principal                       |
| inertial matrix                                                      | Mathematical Concept                      | 37.11.62. principal                       |
| eigenvalues                                                          | Measure/value                             | 37.11.62. principal                       |
| eigenvectors                                                         | Measure/value                             | 37.11.62. principal                       |
| `projection` (cpptraj command)                                       | Method                                    | 37.11.63. projection                      |
| TICA (Time-lagged Independent Component Analysis)                    | Method                                    | 37.11.63. projection; 37.12.39. tica      |
| `pucker` (cpptraj command)                                           | Method                                    | 37.11.64. pucker                          |
| `radgyr` (cpptraj command)                                           | Method                                    | 37.11.65. radgyr \| rog                   |
| radius of gyration                                                   | Measure/value                             | 37.11.65. radgyr \| rog                   |
| radius of gyration tensor                                            | Measure/value                             | 37.11.65. radgyr \| rog                   |
| `radial` (cpptraj command)                                           | Method                                    | 37.11.66. radial \| rdf                   |
| radial distribution function (RDF)                                   | Measure/value                             | 37.11.66. radial \| rdf                   |
| `randomizeions` (cpptraj command)                                    | Method                                    | 37.11.67. randomizeions                   |
| `replicatecell` (cpptraj command)                                    | Method                                    | 37.11.69. replicatecell                   |
| `rmsd` (cpptraj command)                                             | Method                                    | 37.11.70. rms \| rmsd                     |
| RMSD (Root Mean Square Deviation)                                    | Measure/value                             | 37.11.70. rms \| rmsd                     |
| best-fit RMSD                                                        | Measure/value                             | 37.11.70. rms \| rmsd                     |
| per-residue RMSD                                                     | Measure/value                             | 37.11.70. rms \| rmsd                     |
| `rotate` (cpptraj command)                                           | Method                                    | 37.11.74. rotate                          |
| `runavg` (cpptraj command)                                           | Method                                    | 37.11.76. runavg \| runningaverage        |
| `secstruct` (cpptraj command)                                        | Method                                    | 37.11.78. secstruct                       |
| DSSP method                                                          | Method                                    | 37.11.78. secstruct                       |
| Extended beta (secondary structure)                                  | State                                     | 37.11.78. secstruct                       |
| Alpha helix (secondary structure)                                    | State                                     | 37.11.78. secstruct                       |
| `setvelocity` (cpptraj command)                                      | Method                                    | 37.11.79. setvelocity                     |
| Maxwellian distribution                                              | Statistical Model                         | 37.11.79. setvelocity                     |
| SHAKE constraints                                                    | Algorithm / Constraint                    | 37.11.79. setvelocity                     |
| `spam` (cpptraj command)                                             | Method                                    | 37.11.80. spam                            |
| SPAM analysis                                                        | Method                                    | 37.11.80. spam                            |
| `stfcdiffusion` (cpptraj command)                                    | Method                                    | 37.11.81. stfcdiffusion                   |
| MSD (Mean Square Displacement)                                       | Measure/value                             | 37.11.81. stfcdiffusion; 37.12.3. calcdiffusion |
| `strip` (cpptraj command)                                            | Method                                    | 37.11.82. strip                           |
| `surf` (cpptraj command)                                             | Method                                    | 37.11.83. surf                            |
| LCPO algorithm                                                       | Algorithm                                 | 37.11.83. surf                            |
| `symmrmsd` (cpptraj command)                                         | Method                                    | 37.11.84. symmrmsd                        |
| symmetry-corrected RMSD                                              | Measure/value                             | 37.11.84. symmrmsd; 37.12.5. cluster      |
| `temperature` (cpptraj command)                                      | Method                                    | 37.11.85. temperature                     |
| `tordiff` (cpptraj command)                                          | Method                                    | 37.11.87. tordiff                         |
| toroidal-view-preserving (TOR) scheme                                | Method                                    | 37.11.87. tordiff; 37.11.90. unwrap       |
| `unwrap` (cpptraj command)                                           | Method                                    | 37.11.90. unwrap                          |
| fractional coordinates                                               | Convention                                | 37.11.90. unwrap                          |
| `vector` (cpptraj command)                                           | Method                                    | 37.11.91. vector                          |
| dipole vector                                                        | Measure/value                             | 37.11.91. vector                          |
| Debye (unit)                                                         | Unit                                      | 37.11.91. vector                          |
| `velocityautocorr` (cpptraj command)                                 | Method                                    | 37.11.92. velocityautocorr                |
| velocity autocorrelation function (VAC)                              | Measure/value                             | 37.11.92. velocityautocorr                |
| self-diffusion coefficient                                           | Measure/value                             | 37.11.92. velocityautocorr                |
| `volmap` (cpptraj command)                                           | Method                                    | 37.11.93. volmap                          |
| volumetric map                                                       | Data Format                               | 37.11.93. volmap                          |
| `watershell` (cpptraj command)                                       | Method                                    | 37.11.95. watershell                      |
| `xtalsymm` (cpptraj command)                                         | Method                                    | 37.11.96. xtalsymm                        |
| space group                                                          | Convention                                | 37.11.96. xtalsymm                        |
| asymmetric unit                                                      | Concept                                   | 37.11.96. xtalsymm                        |
| `autocorr` (cpptraj analysis command)                                | Method                                    | 37.12.1. autocorr                         |
| `calcdiffusion` (cpptraj analysis command)                           | Method                                    | 37.12.3. calcdiffusion                    |
| `calcstate` (cpptraj analysis command)                               | Method                                    | 37.12.4. calcstate                        |
| `cluster` (cpptraj analysis command)                                 | Method                                    | 37.12.5. cluster                          |
| hierarchical agglomerative clustering                                | Algorithm                                 | 37.12.5. cluster                          |
| DBSCAN                                                               | Algorithm                                 | 37.12.5. cluster                          |
| K-means clustering                                                   | Algorithm                                 | 37.12.5. cluster                          |
| density peaks algorithm                                              | Algorithm                                 | 37.12.5. cluster                          |
| Davies-Bouldin Index (DBI)                                           | Measure/value                             | 37.12.5. cluster                          |
| pseudo-F statistic (pSF)                                             | Measure/value                             | 37.12.5. cluster                          |
| cluster silhouette                                                   | Measure/value                             | 37.12.5. cluster                          |
| `cphstats` (cpptraj analysis command)                                | Method                                    | 37.12.6. cphstats                         |
| `crdfluct` (cpptraj analysis command)                                | Method                                    | 37.12.9. crdfluct                         |
| `curvefit` (cpptraj analysis command)                                | Method                                    | 37.12.11. curvefit                        |
| Levenberg-Marquardt algorithm                                        | Algorithm                                 | 37.12.11. curvefit                        |
| `diagmatrix` (cpptraj analysis command)                              | Method                                    | 37.12.12. diagmatrix                      |
| Principal Component Analysis (PCA)                                   | Method                                    | 37.12.12. diagmatrix; 37.13.1; 37.13.2    |
| Quasiharmonic Analysis                                               | Method                                    | 37.12.12. diagmatrix                      |
| .nmd format                                                          | Data Format                               | 37.12.12. diagmatrix                      |
| `ired` (cpptraj analysis command)                                    | Method                                    | 37.12.19. ired                            |
| `kde` (cpptraj analysis command)                                     | Method                                    | 37.12.20. kde                             |
| Gaussian kernel density estimator                                    | Method                                    | 37.12.20. kde                             |
| `lifetime` (cpptraj analysis command)                                | Method                                    | 37.12.21. lifetime                        |
| `meltcurve` (cpptraj analysis command)                               | Method                                    | 37.12.23. meltcurve                       |
| `modes` (cpptraj analysis command)                                   | Method                                    | 37.12.24. modes                           |
| `remlog` (cpptraj analysis command)                                  | Method                                    | 37.12.30. remlog                          |
| `rmsavgcorr` (cpptraj analysis command)                              | Method                                    | 37.12.32. rmsavgcorr                      |
| RMS average correlation (RAC)                                        | Measure/value                             | 37.12.32. rmsavgcorr                      |
| `rotdif` (cpptraj analysis command)                                  | Method                                    | 37.12.33. rotdif                          |
| diffusion tensor                                                     | Measure/value                             | 37.12.33. rotdif                          |
| `ti` (cpptraj analysis command)                                      | Method                                    | 37.12.38. ti                              |
| Thermodynamic Integration                                            | Method                                    | 37.12.38. ti; 40; 42.1                    |
| Gaussian quadrature                                                  | Method                                    | 37.12.38. ti                              |
| `timecorr` (cpptraj analysis command)                                | Method                                    | 37.12.40. timecorr                        |
| spherical harmonics theory                                           | Theoretical Framework                     | 37.12.40. timecorr                        |
| `wavelet` (cpptraj analysis command)                                 | Method                                    | 37.12.42. wavelet                         |
| continuous wavelet transform (CWT)                                   | Method                                    | 37.12.42. wavelet                         |
| Wavelet Analysis Feature Extraction (WAFEX)                          | Method                                    | 37.12.42. wavelet                         |
| `pytraj`                                                             | Software / API                            | 38. pytraj                                |
| `MMPBSA.py`                                                          | Software / Script                         | 39. MMPBSA.py                             |
| Molecular Mechanics / Poisson Boltzmann (Generalized Born) Surface Area (MM/PB(GB)SA) | Method                                    | 39.1. Introduction                        |
| Poisson Boltzmann (PB) equation                                      | Model / Equation                          | 39.1. Introduction                        |
| Generalized Born (GB) method                                         | Model / Method                            | 39.1. Introduction                        |
| Reference Interaction Site Model (RISM)                              | Model / Method                            | 39.1. Introduction                        |
| APBS (Adaptive Poisson Boltzmann Solver)                             | Software                                  | 39.1. Introduction                        |
| LCPO method (hydrophobic contribution)                               | Algorithm                                 | 39.1. Introduction                        |
| `ante-MMPBSA.py`                                                     | Software / Utility                        | 39.2.2. Using ante-MMPBSA.py              |
| namelist (Amber input format)                                        | Data Format                               | 39.3.1. The input file                    |
| `FEW (Free Energy Workflow)`                                         | Software / Workflow                       | 40. FEW                                   |
| Linear Interaction Energy (LIE) analyses                             | Method                                    | 40. FEW                                   |
| WAMM (Workflow for automated MM-PBSA & MM-GBSA)                      | Module / Workflow                         | 40. FEW; 40.4                             |
| LIEW (Linear interaction energy workflow)                            | Module / Workflow                         | 40. FEW; 40.5                             |
| TIW (Thermodynamic integration workflow)                             | Module / Workflow                         | 40. FEW; 40.6                             |
| RESP charges                                                         | Chemical Property / Method                | 40.3. Common setup... Fig 40.2            |
| AM1-BCC charges                                                      | Chemical Property / Method                | 40.3. Common setup... Fig 40.2            |
| ff12SB force field                                                   | Force Field                               | 40.2. Overview of workflow steps...       |
| Lipid14 force field                                                  | Force Field                               | 40.3.2. Flags for MD setup                |
| CHARMM-GUI Membrane Builder                                          | Software / Tool                           | 40.3.2. Flags for MD setup                |
| soft core potential (TI)                                             | Potential / Method                        | 40.6. Thermodynamic integration workflow |
| Kruskal's algorithm                                                  | Algorithm                                 | 40.6. Thermodynamic integration workflow |
| `BAR/PBSA`                                                           | Method                                    | 41. BAR/PBSA                              |
| `bar_pbsa.py`                                                        | Software / Script                         | 41.2. Usage                               |
| YAML format (bar_pbsa.py input)                                      | Data Format                               | 41.2. Usage                               |
| `edgembar`                                                           | Software / Program                        | 42. edgembar                              |
| MBAR (Multistate Bennett Acceptance Ratio)                           | Method                                    | 42.1. edgembar; 45.1. ndfes               |
| XML format (edgembar input)                                          | Data Format                               | 42.1. edgembar                            |
| `edgembar-amber2dats.py`                                             | Software / Script                         | 42.2. edgembar-amber2dats.py              |
| `fetkutils-tischedule.py`                                            | Software / Script                         | 42.5. fetkutils-tischedule.py             |
| Replica Exchange Acceptance Ratio (AR)                               | Measure/value                             | 42.5. fetkutils-tischedule.py             |
| Phase Space Overlap (PSO)                                            | Measure/value                             | 42.5. fetkutils-tischedule.py             |
| Symmetrized Kullback-Leibler divergence (KL)                         | Measure/value                             | 42.5. fetkutils-tischedule.py             |
| SAXS (Small Angle X-ray Scattering)                                  | Method / Experimental Technique           | 43. SAXS                                  |
| `saxs_rism`                                                          | Software / Program                        | 43.1.1. saxs_rism; 43.2.1. saxs_rism      |
| `saxs_md`                                                            | Software / Program                        | 43.1.2. saxs_md; 43.2.2. saxs_md          |
| dx format (3D-RISM output)                                           | Data Format                               | 43.2.1. saxs_rism                         |
| `MoFT` (analysis of volumetric data)                                 | Software / Library                        | 44. MoFT: analysis of volumetric data     |
| `metatwist`                                                          | Software / Application                    | 44. MoFT: analysis of volumetric data     |
| FFTW library                                                         | Software Library                          | 44. MoFT: analysis of volumetric data     |
| OpenDX format (*.dx)                                                 | Data Format                               | 44.1. Usage                               |
| mrc format                                                           | Data Format                               | 44.1. Usage                               |
| ccp4 format                                                          | Data Format                               | 44.1. Usage                               |
| worm plots                                                           | Data Visualization / Method               | 44.1. Usage                               |
| Laplacian mapping                                                    | Method                                    | 44.2. Examples                            |
| `ndfes`                                                              | Software / Program                        | 45. ndfes                                 |
| vFEP (variational Free Energy Profile)                               | Method                                    | 45.1. ndfes                               |
| wTP (weighted Thermodynamic Perturbation theory)                     | Method                                    | 45.1. ndfes                               |
| FES (Free Energy Surface)                                            | Measure/value                             | 45.1. ndfes                               |
| metafile (ndfes input)                                               | Data Format                               | 45.1. ndfes                               |
| dumpave file (Amber output / ndfes input)                            | Data Format                               | 45.1. ndfes                               |
| `ndfes-PrepareAmberData.py`                                          | Software / Script                         | 45.2. ndfes-PrepareAmberData.py           |
| `ndfes-CheckEquil.py`                                                | Software / Script                         | 45.3. ndfes-CheckEquil.py                 |
| `ndfes-path`                                                         | Software / Script                         | 45.7. ndfes-path                          |
| finite temperature string method                                     | Method                                    | 45.7. ndfes-path                          |
